<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Ribocentre - Publication</title>
  <meta name="description" content="Riboswitches: A riboswitch database">
  <!--<link rel="stylesheet" href="http://localhost:4002/css/main.css>-->
  <link rel="stylesheet" href="/css/main.css">
  <link rel="canonical" href="http://localhost:4002/publications/">
  <link rel="shortcut icon" type ="image/x-icon" href="http://localhost:4002/images/favicon.ico">



</head>


  <body>
    
	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  
<div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <style>
		.navbar-default .navbar-brand {
    color: #ffffff;
		}
        .navbar {
            background-color: #005826;
        }
		#now a{
			color:#fff;
			font-size: 20px;
		}
		#now a:hover{
			font-size: 20px;
		}
	
		
	li {
		float: left;
	}

	li a {
		display: block;
		color: rgb(254, 247, 247);
		text-align: center;
		padding: 14px 16px;
		text-decoration: none;
	}

	li a:hover:not(.active) {
		background-color: #005826;
	}

	li a.active {
		background-color: #000000;
}
</style>



  <div class="container-fluid2">
	<div class="navbar-header">
	  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false">
		<span class="sr-only">Toggle navigation</span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
	  </button>
	
    <a class="navbar-brand" href="https://riboswitch.ribocentre.org/">
		
		<img class="navbar-brand" src="/images/Riboswitch_logo_bggreen.png" style='height:50px;margin-top:-14px;margin-bottom:0;padding:0;font-size: 20px; ' alt="logo">
		&nbsp;Ribocentre-switch 
	</a>
	</div>
	<div class="collapse navbar-collapse" id="navbar-collapse-1">
	  <ul class="nav navbar-nav navbar-right">
		<li id="now" ><a href="http://localhost:4002/">Home</a></li>
		<li id="now"><a href="http://localhost:4002/riboswitches/">Riboswitches</a></li>
		<li id="now"><a href="http://localhost:4002/sequences/">Sequences</a></li>
		<li id="now"><a href="http://localhost:4002/structures/">Structures</a></li>
		<li id="now"><a href="http://localhost:4002/binding_pockets/">Binding pockets</a></li>
		<li id="now"><a href="http://localhost:4002/applications/">Applications</a></li>
		<li id="now"><a href="http://localhost:4002/publications/">Publications</a></li>
		<li id="now"><a href="http://localhost:4002/helps/">Help</a></li>
	  </ul>
	  <script>
		window.addEventListener("DOMContentLoaded", function() {
		  // 获取当前页面的URL
		  var currentURL = window.location.href;
		  
		  // 获取所有导航按钮
		  var navButtons = document.querySelectorAll("li a");
		  
		  // 遍历导航按钮
		  navButtons.forEach(function(button) {
				// 获取按钮的链接
				var buttonURL = button.href;
				
				// 比较URL是否与当前页面的URL匹配
				if (currentURL === buttonURL) {
				  // 设置匹配的按钮为激活状态
				  button.classList.add("active");
				}
		  });
		});
		</script>
	  
				
   </div>
  </div>
</div>
    
      <div class="container-fluid">
        <div class="row">
          <style>
  .box_style{
    background: #f4f4f4;
  }
  .well {
    max-width: 70%;
    margin: 0 auto;
    border-top: 3px solid black;
    }
  .wrapper {
    display: block;
    position: relative;
    width: 100%;
    margin: 0;
    padding: 0;
    text-align: left;
    word-wrap: break-word;
    }
  .sectiontitle {
    display: block;
    max-width: 100%;
    margin: 0 auto ;
    text-align: left;
    background :#fff;
    }
    body {
    padding-top: 70px;
  }
</style>

<div class="wrapper box_style " >
  <br>
  <div class="well" style="border: 1px solid #C9C9C9; background-color: #fff;">
    <section class="sectiontitle"> 
   
     <html>
<head>
	<meta http-equiv="Content-type" content="text/html; charset=utf-8" />
	<meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=no" />
	<title>Ribozyme publication - export</title>
	<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.12.1/css/jquery.dataTables.min.css" />
	<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/buttons/2.2.3/css/buttons.dataTables.min.css" />
</head>
    <style>
		th {
        background-color: #005826;
        color: rgba(255,255,255,0.9);
		    cursor: pointer;
        }
	</style>
    <script type="text/javascript" src="https://code.jquery.com/jquery-3.5.1.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/1.12.1/js/jquery.dataTables.min.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/buttons/2.2.3/js/dataTables.buttons.min.js"></script>
	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.1.3/jszip.min.js"></script>
	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.1.53/pdfmake.min.js"></script>
	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.1.53/vfs_fonts.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/buttons/2.2.3/js/buttons.html5.min.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/buttons/2.2.3/js/buttons.print.min.js"></script>
	<script type="text/javascript">

		$(document).ready(function() {
			$.noConflict();
			$('#mytable').DataTable( {
				dom: 'Bfrtip', 
				buttons: [
					'copy', 'csv', 'excel', 'pdf', 'print'
				]
				
			} );
		} );
    </script>

<h1 class="post-title" itemprop="name headline">Publications</h1>
List of all the articles of Riboswitches.
        
  <body>
    <div>
	  <table id="mytable" class="table table-striped table-bordered" cellspacing="0" width="100%">
    <thead>
      <tr>
        <th>Year</th>
        <th>Author</th>
        <th>Title</th>
        <th>Riboswitch name</th>
        <th>Description</th>
        <th>Journal</th>
      </tr>
    </thead>
        
        <tr>
            <td>updating</td>
            <td>Ames, T. D. &amp; Breaker, R. R. </td>
            <td>Bacterial aptamers that selectively bind glutamine</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glutamine" target="_blank"><b> Glutamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21282981/" target="_blank"><b> RNA Biol. 8, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Ren, A. et al. </td>
            <td>Structural and Dynamic Basis for Low-Affinity, High-Selectivity Binding of L-Glutamine by the Glutamine Riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glutamine" target="_blank"><b> Glutamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/26655897/" target="_blank"><b> Cell Rep. 13, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Klähn, S. et al. </td>
            <td>A glutamine riboswitch is a key element for the regulation of glutamine synthetase in cyanobacteria</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glutamine" target="_blank"><b> Glutamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30085248/" target="_blank"><b> Nucleic Acids Res. 46, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Huang, L., Wang, J., Watkins, A. M., Das, R. &amp; Lilley, D. M. J. </td>
            <td>Structure and ligand binding of the glutamine-II riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glutamine" target="_blank"><b> Glutamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31216023/" target="_blank"><b> Nucleic Acids Res. 47, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sudarsan, N., Wickiser, J. K., Nakamura, S., Ebert, M. S. &amp; Breaker, R. R. </td>
            <td>An mRNA structure in bacteria that controls gene expression by binding lysine</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/14597663/" target="_blank"><b> Genes Dev. 17, 2688–2697 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Grundy, F. J., Lehman, S. C. &amp; Henkin, T. M. </td>
            <td>The L box regulon: lysine sensing by leader RNAs of bacterial lysine biosynthesis genes</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/14523230/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 100, 12057–12062 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Blouin, S. &amp; Lafontaine, D. A. </td>
            <td>A loop loop interaction and a K-turn motif located in the lysine aptamer domain are important for the riboswitch gene regulation control</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/17585050/" target="_blank"><b> RNA 13, 1256–1267 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Blount, K. F., Wang, J. X., Lim, J., Sudarsan, N. &amp; Breaker, R. R. </td>
            <td>Antibacterial lysine analogs that target lysine riboswitches</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/17143270/" target="_blank"><b> Nat. Chem. Biol. 3, 44–49 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Garst, A. D., Héroux, A., Rambo, R. P. &amp; Batey, R. T. </td>
            <td>Crystal structure of the lysine riboswitch regulatory mRNA element</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/18593706/" target="_blank"><b> J. Biol. Chem. 283, 22347–22351 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Serganov, A., Huang, L. &amp; Patel, D. J. </td>
            <td>Structural insights into amino acid binding and gene control by a lysine riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/18784651/" target="_blank"><b> Nature 455, 1263–1267 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Blouin, S., Chinnappan, R. &amp; Lafontaine, D. A. </td>
            <td>Folding of the lysine riboswitch: importance of peripheral elements for transcriptional regulation</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21169337/" target="_blank"><b> Nucleic Acids Res. 39, 3373–3387 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Garst, A. D., Porter, E. B. &amp; Batey, R. T. </td>
            <td>Insights into the regulatory landscape of the lysine riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/22771573/" target="_blank"><b> J. Mol. Biol. 423, 17–33 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Budhathoki, P., Bernal-Perez, L. F., Annunziata, O. &amp; Ryu, Y. </td>
            <td>Rationally-designed fluorescent lysine riboswitch probes</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/22961337/" target="_blank"><b> Org. Biomol. Chem. 10, 7872–7874 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Wilson-Mitchell, S. N., Grundy, F. J. &amp; Henkin, T. M. </td>
            <td>Analysis of lysine recognition and specificity of the Bacillus subtilis L box riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/22416067/" target="_blank"><b> Nucleic Acids Res. 40, 5706–5717 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Zhou, L.-B. &amp; Zeng, A.-P. </td>
            <td>Engineering a Lysine-ON Riboswitch for Metabolic Control of Lysine Production in Corynebacterium glutamicum</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/26300047/" target="_blank"><b> ACS Synth. Biol. 4, 1335–1340 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Zhou, L.-B. &amp; Zeng, A.-P. </td>
            <td>Exploring lysine riboswitch for metabolic flux control and improvement of L-lysine synthesis in Corynebacterium glutamicum</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25575181/" target="_blank"><b> ACS Synth. Biol. 4, 729–734 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Smith-Peter, E., Lamontagne, A.-M. &amp; Lafontaine, D. A. </td>
            <td>Role of lysine binding residues in the global folding of the lysC riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/26403229/" target="_blank"><b> RNA Biol. 12, 1372–1382 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Mukherjee, S., Barash, D. &amp; Sengupta, S. </td>
            <td>Comparative genomics and phylogenomic analyses of lysine riboswitch distributions in bacteria</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28873470/" target="_blank"><b> PLoS One 12, e0184314 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sung, H.-L. &amp; Nesbitt, D. J. </td>
            <td>High pressure single-molecule FRET studies of the lysine riboswitch: cationic and osmolytic effects on pressure induced denaturation</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/32706360/" target="_blank"><b> Phys. Chem. Chem. Phys. 22, 15853–15866 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Marton Menendez, A. &amp; Nesbitt, D. J. </td>
            <td>Ionic Cooperativity between Lysine and Potassium in the Lysine Riboswitch: Single-Molecule Kinetic and Thermodynamic Studies</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Lysine" target="_blank"><b> Lysine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36916791/" target="_blank"><b> J. Phys. Chem. B 127, 2430–2440 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Henkin, T. M., Glass, B. L. &amp; Grundy, F. J. </td>
            <td>Analysis of the Bacillus subtilis tyrS gene: conservation of a regulatory sequence in multiple tRNA synthetase genes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/T-box" target="_blank"><b> T-box</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/1735721/" target="_blank"><b> J. Bacteriol. 174, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Grundy, F. J. &amp; Henkin, T. M. </td>
            <td>tRNA as a positive regulator of transcription antitermination in B. subtilis.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/T-box" target="_blank"><b> T-box</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/8348614/" target="_blank"><b> Cell 74, 475–482 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Yousef, M. R., Grundy, F. J. &amp; Henkin, T. M. </td>
            <td>Structural transitions induced by the interaction between tRNA(Gly) and the Bacillus subtilis glyQS T box leader RNA.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/T-box" target="_blank"><b> T-box</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15890195/" target="_blank"><b> J. Mol. Biol. 349, 273–287 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Zhang, J. &amp; Ferré-D’Amaré, A. R. </td>
            <td>Co-crystal structure of a T-box riboswitch stem I domain in complex with its cognate tRNA.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/T-box" target="_blank"><b> T-box</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/23892783/" target="_blank"><b> Nature 500, 363–366 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sherwood, A. V., Grundy, F. J. &amp; Henkin, T. M. </td>
            <td>T box riboswitches in Actinobacteria: translational regulation via novel tRNA interactions.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/T-box" target="_blank"><b> T-box</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25583497/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 112, 1113–1118 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Battaglia, R. A., Grigg, J. C. &amp; Ke, A. </td>
            <td>Structural basis for tRNA decoding and aminoacylation sensing by T-box riboregulators.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/T-box" target="_blank"><b> T-box</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31740853/" target="_blank"><b> Nat. Struct. Mol. Biol. 26, 1106–1113 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Li, S. et al. </td>
            <td>Structural basis of amino acid surveillance by higher-order tRNA-mRNA interactions.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/T-box" target="_blank"><b> T-box</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31740854/" target="_blank"><b> Nat. Struct. Mol. Biol. 26, 1094–1105 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Gutiérrez-Preciado, A., Henkin, T. M., Grundy, F. J., Yanofsky, C. &amp; Merino, E. </td>
            <td>Biochemical features and functional implications of the RNA-based T-box regulatory mechanism.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/T-box" target="_blank"><b> T-box</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/19258532/" target="_blank"><b> Microbiol. Mol. Biol. Rev. 73, 36–61 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weaver, J. W. &amp; Serganov, A. </td>
            <td>T-box RNA gets boxed.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/T-box" target="_blank"><b> T-box</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31792446/" target="_blank"><b> Nat. Struct. Mol. Biol. 26, 1081–1083 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Gelfand, M. S., Mironov, A. A., Jomantas, J., Kozlov, Y. I. &amp; Perumov, D. A. </td>
            <td>A conserved RNA structure element involved in the regulation of bacterial riboflavin synthesis genes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/10529804/" target="_blank"><b> Trends Genet. 15, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Vitreschak, A. G., Rodionov, D. A., Mironov, A. A. &amp; Gelfand, M. S. </td>
            <td>Regulation of riboflavin biosynthesis and transport genes in bacteria by transcriptional and translational attenuation.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/12136096/" target="_blank"><b> Nucleic Acids Res. 30, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Winkler, W. C., Cohen-Chalamish, S. &amp; Breaker, R. R. </td>
            <td>An mRNA structure that controls gene expression by binding FMN.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/12456892/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 99, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Wickiser, J. K., Winkler, W. C., Breaker, R. R. &amp; Crothers, D. M. </td>
            <td>The speed of RNA transcription and metabolite binding kinetics operate an FMN riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15808508/" target="_blank"><b> Mol. Cell 18, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Lee, E. R., Blount, K. F. &amp; Breaker, R. R. </td>
            <td>Roseoflavin is a natural antibacterial compound that binds to FMN riboswitches and regulates gene expression.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/19246992/" target="_blank"><b> RNA Biol. 6, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Serganov, A., Huang, L. &amp; Patel, D. J. </td>
            <td>Coenzyme recognition and gene regulation by a flavin mononucleotide riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/19169240/" target="_blank"><b> Nature 458, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Vicens, Q., Mondragón, E. &amp; Batey, R. T. </td>
            <td>Molecular sensing by the aptamer domain of the FMN riboswitch: a general model for ligand binding by conformational selection.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21745821/" target="_blank"><b> Nucleic Acids Res. 39, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Blount, K. F. et al. </td>
            <td>Novel riboswitch-binding flavin analog that protects mice against Clostridium difficile infection without inhibiting cecal flora.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/26169403/" target="_blank"><b> Antimicrob. Agents Chemother. 59, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Howe, J. A. et al. </td>
            <td>Selective small-molecule inhibition of an RNA structural element.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/26416753/" target="_blank"><b> Nature 526, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Howe, J. A., L. Xiao, T. O. Fischmann, H. Wang, H. Tang, A. Villafania, R. Zhang, C. M. Barbieri and T. Roemer. </td>
            <td>Atomic resolution mechanistic studies of ribocil: A highly selective unnatural ligand mimic of the E. coli FMN riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/27485612/" target="_blank"><b> RNA Biol. 13, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z., Nelson, J. W., Lünse, C. E., Sherlock, M. E. &amp; Breaker, R. R. </td>
            <td>Bioinformatic analysis of riboswitch structures uncovers variant classes with altered ligand specificity.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28265071/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 114, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Rizvi, N. F. et al. </td>
            <td>Discovery of Selective RNA-Binding Small Molecules by Affinity-Selection Mass Spectrometry.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29412640/" target="_blank"><b> ACS Chem. Biol. 13, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Vicens, Q. et al. </td>
            <td>Structure-Activity Relationship of Flavin Analogues That Target the Flavin Mononucleotide Riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30107111/" target="_blank"><b> ACS Chem. Biol. 13, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Wilt, H. M., Yu, P., Tan, K., Wang, Y. X. &amp; Stagno, J. R. </td>
            <td>FMN riboswitch aptamer symmetry facilitates conformational switching through mutually exclusive coaxial stacking configurations.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/33103111/" target="_blank"><b> Journal of structural biology: X 4, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Harale, B. et al. </td>
            <td>Synthesis and evaluation of antimycobacterial activity of riboflavin derivatives.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/34242760/" target="_blank"><b> Bioorg. Med. Chem. Lett. 48, 128236 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Traykovska, M. &amp; Penchovsky, R. </td>
            <td>Engineering Antisense Oligonucleotides as Antibacterial Agents That Target FMN Riboswitches and Inhibit the Growth of <i>Staphylococcus aureus</i>, <i>Listeria monocytogenes</i>, and <i>Escherichia coli</i>.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/FMN" target="_blank"><b> FMN</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35440139/" target="_blank"><b> ACS Synth. Biol. 11, 1845–1855 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Furukawa, K. et al. </td>
            <td>Bacterial riboswitches cooperatively bind Ni(2+) or Co(2+) ions and control  expression of heavy metal transporters.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/NiCo" target="_blank"><b> NiCo</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25794617/" target="_blank"><b> Mol. Cell 57, 1088–1098 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Xu, J. &amp; Cotruvo, J. A., Jr. </td>
            <td>The (NiCo) Riboswitch Responds to Iron(II).</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/NiCo" target="_blank"><b> NiCo</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/32250107/" target="_blank"><b> Biochemistry 59, 1508–1516 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Xu, J. &amp; Cotruvo, J. A., Jr. </td>
            <td>Iron-responsive riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/NiCo" target="_blank"><b> NiCo</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35427920/" target="_blank"><b> Curr. Opin. Chem. Biol. 68, 102135 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Xu, J. &amp; Cotruvo, J. A., Jr. </td>
            <td>Reconsidering the (NiCo) Riboswitch as an Iron Riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/NiCo" target="_blank"><b> NiCo</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35996475/" target="_blank"><b> ACS Bio Med Chem Au 2, 376–385 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z. et al. </td>
            <td>Identification of 22 candidate structured RNAs in bacteria using the CMfinder comparative genomics pipeline.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-AMP-GMP" target="_blank"><b> c-AMP-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/17621584/" target="_blank"><b> Nucleic Acids Res. 35, 4809–4819 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Nelson, J. W. et al. </td>
            <td>Control of bacterial exoelectrogenesis by c-AMP-GMP.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-AMP-GMP" target="_blank"><b> c-AMP-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25848023/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 112, 5389–5394 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Ren, A. et al. </td>
            <td>Structural basis for molecular discrimination by a 3',3'-cGAMP sensing riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-AMP-GMP" target="_blank"><b> c-AMP-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25818298/" target="_blank"><b> Cell Rep. 11, 1–12 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Kellenberger, C. A. et al. </td>
            <td>GEMM-I riboswitches from Geobacter sense the bacterial second messenger cyclic AMP-GMP.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-AMP-GMP" target="_blank"><b> c-AMP-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25848022/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 112, 5383–5388 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Keller, H., Weickhmann, A. K., Bock, T. &amp; Wöhnert, J. </td>
            <td>Adenine protonation enables cyclic-di-GMP binding to cyclic-GAMP sensing riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-AMP-GMP" target="_blank"><b> c-AMP-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30006500/" target="_blank"><b> RNA 24, 1390–1402 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Li, C., Zhao, X., Zhu, X., Xie, P. &amp; Chen, G. </td>
            <td>Structural Studies of the 3',3'-cGAMP Riboswitch Induced by Cognate and Noncognate Ligands Using Molecular Dynamics Simulation.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-AMP-GMP" target="_blank"><b> c-AMP-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30423927/" target="_blank"><b> Int. J. Mol. Sci. 19, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Tan, Z. et al. </td>
            <td>The Signaling Pathway That cGAMP Riboswitches Found: Analysis and Application of Riboswitches to Study cGAMP Signaling in Geobacter sulfurreducens.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-AMP-GMP" target="_blank"><b> c-AMP-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35163114/" target="_blank"><b> Int. J. Mol. Sci. 23, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Christiansen, L. C., Schou, S., Nygaard, P. &amp; Saxild, H. H. </td>
            <td>Xanthine metabolism in Bacillus subtilis: characterization of the xpt-pbuX operon and evidence for purine- and nitrogen-controlled expression of genes involved in xanthine salvage and catabolism.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Adenine" target="_blank"><b> Adenine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/9098051/" target="_blank"><b> J. Bacteriol. 179, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Johansen, L. E., Nygaard, P., Lassen, C., Agersø, Y. &amp; Saxild, H. H. </td>
            <td>Definition of a second Bacillus subtilis pur regulon comprising the pur and  xpt-pbuX operons plus pbuG, nupG (yxjA), and pbuE (ydhL).</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Adenine" target="_blank"><b> Adenine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/12923093/" target="_blank"><b> J. Bacteriol. 185, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Mandal, M. &amp; Breaker, R. R. </td>
            <td>Adenine riboswitches and gene activation by disruption of a transcription terminator.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Adenine" target="_blank"><b> Adenine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/14718920/" target="_blank"><b> Nat. Struct. Mol. Biol. 11, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Serganov, A. et al. </td>
            <td>Structural Basis for Discriminative Regulation of Gene Expression by Adenine- and Guanine-Sensing mRNAs.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Adenine" target="_blank"><b> Adenine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15610857/" target="_blank"><b> Chem. Biol. 11, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Wickiser, J. K., Cheah, M. T., Breaker, R. R. &amp; Crothers, D. M. </td>
            <td>The Kinetics of Ligand Binding by an Adenine-Sensing Riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Adenine" target="_blank"><b> Adenine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16201765/" target="_blank"><b> Biochemistry 44, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Lemay, J. F., Penedo, J. C., Tremblay, R., Lilley, D. M. &amp; Lafontaine, D. A. </td>
            <td>Folding of the Adenine Riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Adenine" target="_blank"><b> Adenine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16931335/" target="_blank"><b> Chem. Biol. 13, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Delfosse, V. et al. </td>
            <td>Riboswitch structure: an internal residue mimicking the purine ligand.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Adenine" target="_blank"><b> Adenine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20022916/" target="_blank"><b> Nucleic Acids Res. 38, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Frieda, K. L. &amp; Block, S. M. </td>
            <td>Direct observation of cotranscriptional folding in an adenine riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Adenine" target="_blank"><b> Adenine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/23087247/" target="_blank"><b> Science 338, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Stagno, J. R. et al. </td>
            <td>Structures of riboswitch RNA reaction states by mix-and-inject XFEL serial crystallography.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Adenine" target="_blank"><b> Adenine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/27841871/" target="_blank"><b> Nature 541, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>St-Pierre, P. et al. </td>
            <td>A structural intermediate pre-organizes the add adenine riboswitch for ligand recognition.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Adenine" target="_blank"><b> Adenine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/33963862/" target="_blank"><b> Nucleic Acids Res. 49, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Dey, S. K. et al. </td>
            <td>Repurposing an adenine riboswitch into a fluorogenic imaging and sensing tag.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Adenine" target="_blank"><b> Adenine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/34937909/" target="_blank"><b> Nat. Chem. Biol. 18, 180–190 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Reichard, P. </td>
            <td>Interactions between deoxyribonucleotide and DNA synthesis.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/2-dG" target="_blank"><b> 2'-dG</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/3052277/" target="_blank"><b> Annu. Rev. Biochem. 57, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Kim, J. N., Roth, A. &amp; Breaker, R. R. </td>
            <td>Guanine riboswitch variants from Mesoplasma florum selectively recognize  2'-deoxyguanosine.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/2-dG" target="_blank"><b> 2'-dG</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/17911257/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 104, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Pikovskaya, O., Polonskaia, A., Patel, D. J. &amp; Serganov, A. </td>
            <td>Structural principles of nucleoside selectivity in a 2′-deoxyguanosine riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/2-dG" target="_blank"><b> 2'-dG</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21841796/" target="_blank"><b> Nat. Chem. Biol. 7, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z., Nelson, J. W., Lünse, C. E., Sherlock, M. E. &amp; Breaker, R. R. </td>
            <td>Bioinformatic analysis of riboswitch structures uncovers variant classes with altered ligand specificity.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/2-dG" target="_blank"><b> 2'-dG</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28265071/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 114, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Matyjasik, M. M. &amp; Batey, R. T. </td>
            <td>Structural basis for 2′-deoxyguanosine recognition by the 2′-dG-II class of riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/2-dG" target="_blank"><b> 2'-dG</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31598729/" target="_blank"><b> Nucleic Acids Res. 47, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Hamal, D. S., Panchapakesan, S. S. S., Slattery, P., Roth, A. &amp; Breaker, R. R. </td>
            <td>Variants of the guanine riboswitch class exhibit altered ligand specificities for xanthine, guanine, or 2′-deoxyguanosine.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/2-dG" target="_blank"><b> 2'-dG</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35622895/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 119, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Wang, J. X., Lee, E. R., Morales, D. R., Lim, J., &amp; Breaker, R. R. </td>
            <td>Riboswitches that sense S-adenosylhomocysteine and activate genes involved in coenzyme recycling.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAH" target="_blank"><b> SAH</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/18374645/" target="_blank"><b> Mol. Cell 29, 691–702 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Wang, J. X., &amp; Breaker, R. R. </td>
            <td>Riboswitches that sense S-adenosylmethionine and S-adenosylhomocysteine.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAH" target="_blank"><b> SAH</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/18443629/" target="_blank"><b> Biochem. Cell Biol. 86, 157–168 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Edwards, A. L., Reyes, F. E., Héroux, A., &amp; Batey, R. T. </td>
            <td>Structural basis for recognition of S-adenosylhomocysteine by riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAH" target="_blank"><b> SAH</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20864509/" target="_blank"><b> RNA 16, 2144–2155 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Chou, M. Y., Lin, S. C., &amp; Chang, K. Y. </td>
            <td>Stimulation of -1 programmed ribosomal frameshifting by a metabolite-responsive RNA pseudoknot.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAH" target="_blank"><b> SAH</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20435898/" target="_blank"><b> RNA 16, 1236–1244 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Zheng L, Song Q, Xu X, Shen X, Li C, Li H, Chen H, Ren A. </td>
            <td>Structure-based insights into recognition and regulation of SAM-sensing riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAH" target="_blank"><b> SAH</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36459353/" target="_blank"><b> Sci China Life Sci.66</b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z., Wang, J. X., Bogue, J., Yang, J., Corbino, K., Moy, R. H., &amp; Breaker, R. R. </td>
            <td>Comparative genomics reveals 104 candidate structured RNAs from bacteria, archaea, and their metagenomes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-SAH" target="_blank"><b> SAM-SAH</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20230605/" target="_blank"><b> Genome Biol. 11, R31 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z., Lünse, C. E., Corbino, K. A., Ames, T. D., Nelson, J. W., Roth, A., Perkins, K. R., Sherlock, M. E., &amp; Breaker, R. R. </td>
            <td>Detection of 224 candidate structured RNAs by comparative analysis of specific subsets of intergenic regions.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-SAH" target="_blank"><b> SAM-SAH</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28977401/" target="_blank"><b> Nucleic Acids Res. 45, 10811–10823 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weickhmann, A. K., Keller, H., Duchardt-Ferner, E., Strebitzer, E., Juen, M. A., Kremser, J., Wurm, J. P., Kreutz, C., &amp; Wöhnert, J. </td>
            <td>NMR resonance assignments for the SAM/SAH-binding riboswitch RNA bound to S-adenosylhomocysteine.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-SAH" target="_blank"><b> SAM-SAH</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30051308/" target="_blank"><b> Biomol. NMR Assign. 12, 329–334 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weickhmann, A. K., Keller, H., Wurm, J. P., Strebitzer, E., Juen, M. A., Kremser, J., Weinberg, Z., Kreutz, C., Duchardt-Ferner, E., &amp; Wöhnert, J. </td>
            <td>The structure of the SAM/SAH-binding riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-SAH" target="_blank"><b> SAM-SAH</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30590743/" target="_blank"><b> Nucleic Acids Res. 47, 2654–2665 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Huang, L., Liao, T. W., Wang, J., Ha, T., &amp; Lilley, D. M. J. </td>
            <td>Crystal structure and ligand-induced folding of the SAM/SAH riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-SAH" target="_blank"><b> SAM-SAH</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/32520325/" target="_blank"><b> Nucleic Acids Res. 48, 7545–7556 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Hu, G., &amp; Zhou, H. X. </td>
            <td>Magnesium ions mediate ligand binding and conformational transition of the SAM/SAH riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-SAH" target="_blank"><b> SAM-SAH</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36945415/" target="_blank"><b> BioRxiv : the preprint server for biology, 2023.03.12.532287</b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Zheng L, Song Q, Xu X, Shen X, Li C, Li H, Chen H, Ren A. </td>
            <td>Structure-based insights into recognition and regulation of SAM-sensing riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-SAH" target="_blank"><b> SAM-SAH</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36459353/" target="_blank"><b> Sci China Life Sci.66</b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z. et al. </td>
            <td>Comparative genomics reveals 104 candidate structured RNAs from bacteria, archaea, and their metagenomes</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ZTP" target="_blank"><b> ZTP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20230605/" target="_blank"><b> Genome Biol. 11, R31 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Kim, P. B., Nelson, J. W. &amp; Breaker, R. R. </td>
            <td>An ancient riboswitch class in bacteria regulates purine biosynthesis and one-carbon metabolism</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ZTP" target="_blank"><b> ZTP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25616067/" target="_blank"><b> Mol. Cell 57, 317–328 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Ren, A., Rajashankar, K. R. &amp; Patel, D. J. </td>
            <td>Global RNA Fold and Molecular Recognition for a pfl Riboswitch Bound to ZMP, a Master Regulator of One-Carbon Metabolism</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ZTP" target="_blank"><b> ZTP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/26118534/" target="_blank"><b> Structure 23, 1375–1381 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Trausch, J. J., Marcano-Velázquez, J. G., Matyjasik, M. M. &amp; Batey, R. T. </td>
            <td>Metal Ion-Mediated Nucleobase Recognition by the ZTP Riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ZTP" target="_blank"><b> ZTP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/26144884/" target="_blank"><b> Chem. Biol. 22, 829–837 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Jones, C. P. &amp; Ferré-D’Amaré, A. R. </td>
            <td>Recognition of the bacterial alarmone ZMP through long-distance association of two RNA subdomains</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ZTP" target="_blank"><b> ZTP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/26280533/" target="_blank"><b> Nat. Struct. Mol. Biol. 22, 679–685 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Jones, C. et al. </td>
            <td>Co-crystal structure of the Fusobacterium ulcerans ZTP riboswitch using an X-ray free-electron laser</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ZTP" target="_blank"><b> ZTP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31282869/" target="_blank"><b> Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun. 75, 496–500 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Strobel, E. J., Cheng, L., Berman, K. E., Carlson, P. D. &amp; Lucks, J. B. </td>
            <td>A ligand-gated strand displacement mechanism for ZTP riboswitch transcription control</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ZTP" target="_blank"><b> ZTP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31636437/" target="_blank"><b> Nat. Chem. Biol. 15, 1067–1076 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Tran, B. et al. </td>
            <td>Parallel Discovery Strategies Provide a Basis for Riboswitch Ligand Design</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ZTP" target="_blank"><b> ZTP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/32795418/" target="_blank"><b> Cell Chem Biol 27, 1241–1249.e4 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Hua, B. et al. </td>
            <td>Real-time monitoring of single ZTP riboswitches reveals a complex and kinetically controlled decision landscape</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ZTP" target="_blank"><b> ZTP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/32913225/" target="_blank"><b> Nat. Commun. 11, 4531 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Bushhouse, D. Z. &amp; Lucks, J. B. </td>
            <td>Tuning strand displacement kinetics enables programmable ZTP riboswitch dynamic range in vivo</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ZTP" target="_blank"><b> ZTP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36864761/" target="_blank"><b> Nucleic Acids Res. 51, 2891–2903 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Perkins, K. R., Atilho, R. M., Moon, M. H. &amp; Breaker, R. R. </td>
            <td>Employing a ZTP Riboswitch to Detect Bacterial Folate Biosynthesis Inhibitors in a Small Molecule High-Throughput Screen</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ZTP" target="_blank"><b> ZTP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31609568/" target="_blank"><b> ACS Chem. Biol. 14, 2841–2850 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Nelson, J. W., Atilho, R. M., Sherlock, M. E., Stockbridge, R. B. &amp; Breaker, R. R. </td>
            <td>Metabolism of Free Guanidine in Bacteria Is Regulated by a Widespread Riboswitch Class.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanidine" target="_blank"><b> Guanidine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/27989440/" target="_blank"><b> Mol. Cell 65, 220–230 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sherlock, M. E., Malkowski, S. N. &amp; Breaker, R. R. </td>
            <td>Biochemical Validation of a Second Guanidine Riboswitch Class in Bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanidine" target="_blank"><b> Guanidine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28001368/" target="_blank"><b> Biochemistry 56, 352–358 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sherlock, M. E. &amp; Breaker, R. R. </td>
            <td>Biochemical Validation of a Third Guanidine Riboswitch Class in Bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanidine" target="_blank"><b> Guanidine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28001372/" target="_blank"><b> Biochemistry 56, 359–363 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Reiss, C. W., Xiong, Y. &amp; Strobel, S. A. </td>
            <td>Structural Basis for Ligand Binding to the Guanidine-I Riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanidine" target="_blank"><b> Guanidine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28017522/" target="_blank"><b> Structure 25, 195–202 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Battaglia, R. A., Price, I. R. &amp; Ke, A. </td>
            <td>ykkCStructural basis for guanidine sensing by the family of riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanidine" target="_blank"><b> Guanidine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28096518/" target="_blank"><b> RNA 23, 578–585 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Huang, L., Wang, J. &amp; Lilley, D. M. J. </td>
            <td>The Structure of the Guanidine-II Riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanidine" target="_blank"><b> Guanidine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28529131/" target="_blank"><b> Cell Chem Biol 24, 695–702.e2 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Reiss, C. W. &amp; Strobel, S. A. </td>
            <td>Structural basis for ligand binding to the guanidine-II riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanidine" target="_blank"><b> Guanidine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28600356/" target="_blank"><b> RNA 23, 1338–1343 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Huang, L., Wang, J., Wilson, T. J. &amp; Lilley, D. M. J. </td>
            <td>Structure of the Guanidine III Riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanidine" target="_blank"><b> Guanidine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28988949/" target="_blank"><b> Cell Chem Biol 24, 1407–1415.e2 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Huang, L., Wang, J., Wilson, T. J. &amp; Lilley, D. M. J.  </td>
            <td>Structure-guided design of a high-affinity ligand for a riboswitch. </td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanidine" target="_blank"><b> Guanidine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30609994/" target="_blank"><b> RNA 25, 423–430 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Wuebben, C., Vicino, M. F., Mueller, M. &amp; Schiemann, O. </td>
            <td>Do the P1 and P2 hairpins of the Guanidine-II riboswitch interact?</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanidine" target="_blank"><b> Guanidine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/32857846/" target="_blank"><b> Nucleic Acids Res. 48, 10518–10526 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Salvail, H., Balaji, A., Yu, D., Roth, A. &amp; Breaker, R. R. </td>
            <td>Biochemical Validation of a Fourth Guanidine Riboswitch Class in Bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanidine" target="_blank"><b> Guanidine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/33236895/" target="_blank"><b> Biochemistry 59, 4654–4662 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Lenkeit, F., Eckert, I., Hartig, J. S. &amp; Weinberg, Z. </td>
            <td>Discovery and characterization of a fourth class of guanidine riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanidine" target="_blank"><b> Guanidine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/33237283/" target="_blank"><b> Nucleic Acids Res. 48, 12889–12899 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sinn, M., Hauth, F., Lenkeit, F., Weinberg, Z. &amp; Hartig, J. S. </td>
            <td>Widespread bacterial utilization of guanidine as nitrogen source.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanidine" target="_blank"><b> Guanidine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/33590553/" target="_blank"><b> Mol. Microbiol. 116, 200–210 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Fuks, C., Falkner, S., Schwierz, N. &amp; Hengesbach, M. </td>
            <td>Combining Coarse-Grained Simulations and Single Molecule Analysis Reveals a Three-State Folding Model of the Guanidine-II Riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanidine" target="_blank"><b> Guanidine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35573739/" target="_blank"><b> Front Mol Biosci 9, 826505 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Mirihana Arachchilage, G., Sherlock, M. E., Weinberg, Z., &amp; Breaker, R. R. </td>
            <td>SAM-VI RNAs selectively bind S-adenosylmethionine and exhibit similarities to SAM-III riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-VI" target="_blank"><b> SAM-VI</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29106323/" target="_blank"><b> RNA Biol. 15, 371–378 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sun, A., Gasser, C., Li, F., Chen, H., Mair, S., Krasheninina, O., Micura, R., &amp; Ren, A. </td>
            <td>SAM-VI riboswitch structure and signature for ligand discrimination.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-VI" target="_blank"><b> SAM-VI</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31844059/" target="_blank"><b> Nat. Commun. 10, 5728 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Xue, Y., Li, J., Chen, D., Zhao, X., Hong, L., &amp; Liu, Y. </td>
            <td>Observation of structural switch in nascent SAM-VI riboswitch during transcription at single-nucleotide and single-molecule resolution.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-VI" target="_blank"><b> SAM-VI</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/37087479/" target="_blank"><b> Nat. Commun. 14, 2320 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Zheng L, Song Q, Xu X, Shen X, Li C, Li H, Chen H, Ren A. </td>
            <td>Structure-based insights into recognition and regulation of SAM-sensing riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-VI" target="_blank"><b> SAM-VI</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36459353/" target="_blank"><b> Sci China Life Sci.66</b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z. et al. </td>
            <td>Detection of 224 candidate structured RNAs by comparative analysis of specific subsets of intergenic regions.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PRA" target="_blank"><b> PRA</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28977401/" target="_blank"><b> Nucleic Acids Res. 45, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Malkowski, S. N., Atilho, R. M., Greenlee, E. B., Weinberg, C. E. &amp; Breaker, R. R. </td>
            <td>A rare bacterial RNA motif is implicated in the regulation of the purF gene whose encoded enzyme synthesizes phosphoribosylamine.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PRA" target="_blank"><b> PRA</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/32843366/" target="_blank"><b> RNA 26, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Barrick, J. E. et al. </td>
            <td>New RNA motifs suggest an expanded scope for riboswitches in bacterial genetic control.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ppGpp" target="_blank"><b> ppGpp</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15096624/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 101, 6421–6426 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Nelson, J. W., Atilho, R. M., Sherlock, M. E., Stockbridge, R. B. &amp; Breaker, R. R. </td>
            <td>Metabolism of Free Guanidine in Bacteria Is Regulated by a Widespread Riboswitch Class.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ppGpp" target="_blank"><b> ppGpp</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/27989440/" target="_blank"><b> Mol. Cell 65, 220–230 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sherlock, M. E., Sudarsan, N., Stav, S. &amp; Breaker, R. R. </td>
            <td>Tandem riboswitches form a natural Boolean logic gate to control purine metabolism in bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ppGpp" target="_blank"><b> ppGpp</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29504937/" target="_blank"><b> Elife 7, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sherlock, M. E., Sudarsan, N. &amp; Breaker, R. R. </td>
            <td>Riboswitches for the alarmone ppGpp expand the collection of RNA-based signaling systems.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ppGpp" target="_blank"><b> ppGpp</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29784782/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 115, 6052–6057 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Peselis, A. &amp; Serganov, A. </td>
            <td>ykkC riboswitches employ an add-on helix to adjust specificity for polyanionic ligands. </td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ppGpp" target="_blank"><b> ppGpp</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30120360/" target="_blank"><b> Nat. Chem. Biol. 14, 887–894 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sun, Z. et al. </td>
            <td>Live-Cell Imaging of Guanosine Tetra- and Pentaphosphate (p)ppGpp with RNA-based Fluorescent Sensors*.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ppGpp" target="_blank"><b> ppGpp</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/34487413/" target="_blank"><b> Angew. Chem. Int. Ed Engl. 60, 24070–24074 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Barrick, J. E. et al. </td>
            <td>New RNA motifs suggest an expanded scope for riboswitches in bacterial genetic control.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PRPP" target="_blank"><b> PRPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15096624/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 101, 6421–6426 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Nelson, J. W., Atilho, R. M., Sherlock, M. E., Stockbridge, R. B. &amp; Breaker, R. R. </td>
            <td>Metabolism of Free Guanidine in Bacteria Is Regulated by a Widespread Riboswitch Class.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PRPP" target="_blank"><b> PRPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/27989440/" target="_blank"><b> Mol. Cell 65, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sherlock, M. E., Sudarsan, N., Stav, S. &amp; Breaker, R. R. </td>
            <td>Tandem riboswitches form a natural Boolean logic gate to control purine metabolism in bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PRPP" target="_blank"><b> PRPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29504937/" target="_blank"><b> Elife 7, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Knappenberger, A. J., Reiss, C. W. &amp; Strobel, S. A. </td>
            <td>Structures of two aptamers with differing ligand specificity reveal ruggedness in the functional landscape of RNA.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PRPP" target="_blank"><b> PRPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29877798/" target="_blank"><b> Elife 7, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Peselis, A. &amp; Serganov, A. </td>
            <td>ykkC riboswitches employ an add-on helix to adjust specificity for polyanionic ligands.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PRPP" target="_blank"><b> PRPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30120360/" target="_blank"><b> Nat. Chem. Biol. 14, 887–894 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Christiansen, L. C., Schou, S., Nygaard, P. &amp; Saxild, H. H. </td>
            <td>Xanthine metabolism in Bacillus subtilis: characterization of the xpt-pbuX operon and evidence for purine- and nitrogen-controlled expression of genes involved in xanthine salvage and catabolism.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanine" target="_blank"><b> Guanine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/9098051/" target="_blank"><b> J. Bacteriol. 179, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Mandal, M., Boese, B., Barrick, J. E., Winkler, W. C. &amp; Breaker, R. R. </td>
            <td>Riboswitches Control Fundamental Biochemical Pathways in Bacillus subtilis and Other Bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanine" target="_blank"><b> Guanine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/12787499/" target="_blank"><b> Cell 113, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Batey, R. T., Gilbert, S. D. &amp; Montange, R. K. </td>
            <td>Structure of a natural guanine-responsive riboswitch complexed with the  metabolite hypoxanthine.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanine" target="_blank"><b> Guanine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15549109/" target="_blank"><b> Nature 432, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Serganov, A. et al. </td>
            <td>Structural Basis for Discriminative Regulation of Gene Expression by Adenine- and Guanine-Sensing mRNAs.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanine" target="_blank"><b> Guanine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15610857/" target="_blank"><b> Chem. Biol. 11, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Gilbert, S. D., Stoddard, C. D., Wise, S. J. &amp; Batey, R. T. </td>
            <td>Thermodynamic and Kinetic Characterization of Ligand Binding to the Purine Riboswitch Aptamer Domain.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanine" target="_blank"><b> Guanine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16650860/" target="_blank"><b> J. Mol. Biol. 359, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Gilbert, S. D., Reyes, F. E., Edwards, A. L. &amp; Batey, R. T. </td>
            <td>Adaptive Ligand Binding by the Purine Riboswitch in the Recognition of Guanine and Adenine Analogs.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanine" target="_blank"><b> Guanine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/19523903/" target="_blank"><b> Structure 17, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Buck, J. et al. </td>
            <td>Influence of ground-state structure and Mg 2+ binding on folding kinetics of the guanine-sensing riboswitch aptamer domain.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanine" target="_blank"><b> Guanine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21890900/" target="_blank"><b> Nucleic Acids Res. 39, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Hernandez, A. R. et al. </td>
            <td>A Crystal Structure of a Functional RNA Molecule Containing an Artificial Nucleobase Pair.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanine" target="_blank"><b> Guanine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/26223188/" target="_blank"><b> Angew. Chem. Int. Ed Engl. 54, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Hamal, D. S., Panchapakesan, S. S. S., Slattery, P., Roth, A. &amp; Breaker, R. R. </td>
            <td>Variants of the guanine riboswitch class exhibit altered ligand specificities for xanthine, guanine, or 2′-deoxyguanosine.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Guanine" target="_blank"><b> Guanine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35622895/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 119, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Schwarz, G. </td>
            <td>Molybdenum cofactor biosynthesis and deficiency.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/MoCo&amp;Wco" target="_blank"><b> MoCo&amp;Wco</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16261263/" target="_blank"><b> Cell. Mol. Life Sci. 62, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z. et al. </td>
            <td>Identification of 22 candidate structured RNAs in bacteria using the CMfinder comparative genomics pipeline.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/MoCo&amp;Wco" target="_blank"><b> MoCo&amp;Wco</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/17621584/" target="_blank"><b> Nucleic Acids Res. 35, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Regulski, E. E. et al. </td>
            <td>A widespread riboswitch candidate that controls bacterial genes involved in molybdenum cofactor and tungsten cofactor metabolism.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/MoCo&amp;Wco" target="_blank"><b> MoCo&amp;Wco</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/18363797/" target="_blank"><b> Mol. Microbiol. 68, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Bevers, L. E. et al. </td>
            <td>Function of MoaB proteins in the biosynthesis of the molybdenum and tungsten cofactors.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/MoCo&amp;Wco" target="_blank"><b> MoCo&amp;Wco</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/18154309/" target="_blank"><b> Biochemistry 47, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Iobbi-Nivol, C. &amp; Leimkühler, . </td>
            <td>Molybdenum enzymes, their maturation and molybdenum cofactor biosynthesis in Escherichia coli.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/MoCo&amp;Wco" target="_blank"><b> MoCo&amp;Wco</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/23201473/" target="_blank"><b> Biochim. Biophys. Acta 1827, 1086–1101 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Hover, B. M., Tonthat, N. K., Schumacher, M. A. &amp; Yokoyama, K. </td>
            <td>Mechanism of pyranopterin ring formation in molybdenum cofactor biosynthesis.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/MoCo&amp;Wco" target="_blank"><b> MoCo&amp;Wco</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25941396/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 112, 6347–6352 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Pavlova, N. &amp; Penchovsky, R. </td>
            <td>Bioinformatics and Genomic Analyses of the Suitability of Eight Riboswitches for Antibacterial Drug Targets.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/MoCo&amp;Wco" target="_blank"><b> MoCo&amp;Wco</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36139956/" target="_blank"><b> Antibiotics </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Amadei, F., Reichenbach, M., Gallo, S. &amp; Sigel, R. K. O. </td>
            <td>The structural features of the ligand-free moaA riboswitch and its ion-dependent folding.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/MoCo&amp;Wco" target="_blank"><b> MoCo&amp;Wco</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36774787/" target="_blank"><b> J. Inorg. Biochem. 242, 112153 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Barrick, J. E. et al. </td>
            <td>New RNA motifs suggest an expanded scope for riboswitches in bacterial genetic control.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Magnesium" target="_blank"><b> Magnesium</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15096624/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 101, 6421–6426 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Cromie, M. J., Shi, Y., Latifi, T. &amp; Groisman, E. A. </td>
            <td>An RNA sensor for intracellular Mg(2+).</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Magnesium" target="_blank"><b> Magnesium</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16615891/" target="_blank"><b> Cell 125, 71–84 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Dann, C. E., 3rd et al. </td>
            <td>Structure and mechanism of a metal-sensing regulatory RNA.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Magnesium" target="_blank"><b> Magnesium</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/17803910/" target="_blank"><b> Cell 130, 878–892 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Wakeman, C. A., Ramesh, A. &amp; Winkler, W. C. </td>
            <td>Multiple metal-binding cores are required for metalloregulation by M-box riboswitch RNAs.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Magnesium" target="_blank"><b> Magnesium</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/19619558/" target="_blank"><b> J. Mol. Biol. 392, 723–735 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Ramesh, A., Wakeman, C. A. &amp; Winkler, W. C. </td>
            <td>Insights into metalloregulation by M-box riboswitch RNAs via structural analysis of manganese-bound complexes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Magnesium" target="_blank"><b> Magnesium</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21315082/" target="_blank"><b> J. Mol. Biol. 407, 556–570 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Pavlova, N. &amp; Penchovsky, R. </td>
            <td>Bioinformatics and Genomic Analyses of the Suitability of Eight Riboswitches for Antibacterial Drug Targets.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Magnesium" target="_blank"><b> Magnesium</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36139956/" target="_blank"><b> Antibiotics </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Li, S., Hwang, X. Y., Stav, S. &amp; Breaker, R. R. </td>
            <td>The yjdF riboswitch candidate regulates gene expression by binding diverse azaaromatic compounds.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Azaaromatic" target="_blank"><b> Azaaromatic</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/26843526/" target="_blank"><b> RNA 22, 530–541 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Gong, S., Wang, Y., Wang, Z., Wang, Y. &amp; Zhang, W. </td>
            <td>Genetic regulation mechanism of the yjdF riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Azaaromatic" target="_blank"><b> Azaaromatic</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29223402/" target="_blank"><b> J. Theor. Biol. 439, 152–159 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Trachman, R. J., 3rd, Passalacqua, L. F. M. &amp; Ferré-D’Amaré, A. R. </td>
            <td>The bacterial yjdF riboswitch regulates translation through its tRNA-like fold.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Azaaromatic" target="_blank"><b> Azaaromatic</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35427649/" target="_blank"><b> J. Biol. Chem. 298, 101934 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z. et al. </td>
            <td>Detection of 224 candidate structured RNAs by comparative analysis of specific subsets of intergenic regions.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Xanthine" target="_blank"><b> Xanthine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28977401/" target="_blank"><b> Nucleic Acids Res. 45, 10811–10823 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Yu, D. &amp; Breaker, R. R. </td>
            <td>A bacterial riboswitch class senses xanthine and uric acid to regulate genes associated with purine oxidation.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Xanthine" target="_blank"><b> Xanthine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/32345632/" target="_blank"><b> RNA 26, 960–968 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Xu, X. et al. </td>
            <td>Insights into xanthine riboswitch structure and metal ion-mediated ligand recognition.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Xanthine" target="_blank"><b> Xanthine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/34125892/" target="_blank"><b> Nucleic Acids Res. 49, 7139–7153 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Hamal Dhakal, S., Panchapakesan, S. S. S., Slattery, P., Roth, A. &amp; Breaker, R. R. </td>
            <td>Variants of the guanine riboswitch class exhibit altered ligand specificities for xanthine, guanine, or 2'-deoxyguanosine.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Xanthine" target="_blank"><b> Xanthine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35622895/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 119, e2120246119 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Winkler, W. C., Nahvi, A., Roth, A., Collins, J. A. &amp; Breaker, R. R. </td>
            <td>Control of gene expression by a natural metabolite-responsive ribozyme.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/GlcN6P" target="_blank"><b> GlcN6P</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15029187" target="_blank"><b> Nature 428, 281–286 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Knudsen, S. M. &amp; Ellington, A. D. </td>
            <td>Ribozyme déjà vu.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/GlcN6P" target="_blank"><b> GlcN6P</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15048102" target="_blank"><b> Nat. Struct. Mol. Biol. 11, 301–303 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Jansen, J. A., McCarthy, T. J., Soukup, G. A. &amp; Soukup, J. K. </td>
            <td>Backbone and nucleobase contacts to glucosamine-6-phosphate in the glmS ribozyme.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/GlcN6P" target="_blank"><b> GlcN6P</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16699515" target="_blank"><b> Nat. Struct. Mol. Biol. 13, 517–523 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Klein, D. J. &amp; Ferré-D’Amaré, A. R. </td>
            <td>Structural basis of glmS ribozyme activation by glucosamine-6-phosphate.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/GlcN6P" target="_blank"><b> GlcN6P</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16990543" target="_blank"><b> Science 313, 1752–1756 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Roth, A., Nahvi, A., Lee, M., Jona, I. &amp; Breaker, R. R. </td>
            <td>Characteristics of the glmS ribozyme suggest only structural roles for divalent metal ions.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/GlcN6P" target="_blank"><b> GlcN6P</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16484375" target="_blank"><b> Rna 12</b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Soukup, G. A. </td>
            <td>Core requirements for glmS ribozyme self-cleavage reveal a putative pseudoknot structure.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/GlcN6P" target="_blank"><b> GlcN6P</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16464827" target="_blank"><b> Nucleic Acids Res. 34, 968–975 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Cochrane, J. C., Lipchock, S. V., Smith, K. D. &amp; Strobel, S. A. </td>
            <td>Structural and chemical basis for glucosamine 6-phosphate binding and activation of the glmS ribozyme.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/GlcN6P" target="_blank"><b> GlcN6P</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/19228039" target="_blank"><b> Biochemistry 48, 3239–3246 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Watson, P. Y. &amp; Fedor, M. J. </td>
            <td>The glmS riboswitch integrates signals from activating and inhibitory metabolites in vivo.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/GlcN6P" target="_blank"><b> GlcN6P</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21317896" target="_blank"><b> Nat. Struct. Mol. Biol. 18, 359–363 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Viladoms, J. &amp; Fedor, M. J. </td>
            <td>The glmS ribozyme cofactor is a general acid-base catalyst.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/GlcN6P" target="_blank"><b> GlcN6P</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/23113700" target="_blank"><b> J. Am. Chem. Soc. 134, 19043–19049 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Bingaman, J. L. et al. </td>
            <td>The GlcN6P cofactor plays multiple catalytic roles in the glmS ribozyme.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/GlcN6P" target="_blank"><b> GlcN6P</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28192411" target="_blank"><b> Nat. Chem. Biol. 13, 439–445 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Pavlova, N. &amp; Penchovsky, R. </td>
            <td>Genome-wide bioinformatics analysis of FMN, SAM-I, glmS, TPP, lysine, purine, cobalamin, and SAH riboswitches for their applications as allosteric antibacterial drug targets in human pathogenic bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/GlcN6P" target="_blank"><b> GlcN6P</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31079546" target="_blank"><b> Expert Opin. Ther. Targets 23, 631–643 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Traykovska, M., Popova, K. B. &amp; Penchovsky, R. </td>
            <td>Targeting glmS Ribozyme with Chimeric Antisense Oligonucleotides for Antibacterial Drug Development.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/GlcN6P" target="_blank"><b> GlcN6P</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/34734706" target="_blank"><b> ACS Synth. Biol. 10, 3167–3176 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Begley, T. P. et al. </td>
            <td>Thiamin biosynthesis in prokaryotes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/10382260/" target="_blank"><b> Arch. Microbiol. 171, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Miranda-Ríos, J., Navarro, M. &amp; Soberón, M. </td>
            <td>A conserved RNA structure (thi box) is involved in regulation of thiamin biosynthetic gene expression in bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/11470904/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 98, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Winkler, W., Nahvi, A. &amp; Breaker, R. R. </td>
            <td>Thiamine derivatives bind messenger RNAs directly to regulate bacterial gene expression.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/12410317/" target="_blank"><b> Nature 419, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sudarsan, N., Barrick, J. E. &amp; Breaker, R. R. </td>
            <td>Metabolite-binding RNA domains are present in the genes of eukaryotes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/12756322/" target="_blank"><b> RNA 9, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Kubodera, T., M. Watanabe, K. Yoshiuchi, N. Yamashita, A. Nishimura, S. Nakai, K. Gomi and H. Hanamoto. </td>
            <td>Thiamine-regulated gene expression of Aspergillus oryzae thiA requires splicing of the intron containing a riboswitch-like domain in the 5'-UTR</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/14675766/" target="_blank"><b> FEBS Lett 555</b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Yamauchi, T. et al. </td>
            <td>Roles of Mg<sup>2+</sup> in TPP-dependent riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15862294/" target="_blank"><b> FEBS Lett. 579, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sudarsan, N., Cohen-Chalamish, S., Nakamura, S., Emilsson, G. M. &amp; Breaker, R. R. </td>
            <td>Thiamine pyrophosphate riboswitches are targets for the antimicrobial compound pyrithiamine.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16356850/" target="_blank"><b> Chem. Biol. 12, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Thore, S., Leibundgut, M. &amp; Ban, N. </td>
            <td>Structure of the eukaryotic thiamine pyrophosphate riboswitch with its regulatory ligand.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16675665/" target="_blank"><b> Science 312, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Serganov, A., Polonskaia, A., Phan, A. T., Breaker, R. R. &amp; Patel, D. J. </td>
            <td>Structural basis for gene regulation by a thiamine pyrophosphate-sensing riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16728979/" target="_blank"><b> Nature 441, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Edwards, T. E. &amp; Ferré-D’Amaré, A. R. </td>
            <td>Crystal Structures of the Thi-Box Riboswitch Bound to Thiamine Pyrophosphate Analogs Reveal Adaptive RNA-Small Molecule. Recognition</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16962976/" target="_blank"><b> Structure 14, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Bocobza, S. et al. </td>
            <td>Riboswitch-dependent gene regulation and its evolution in the plant kingdom.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/18006684/" target="_blank"><b> Genes Dev. 21, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Cheah, M. T., Wachter, A., Sudarsan, N. &amp; Breaker, R. R. </td>
            <td>Control of alternative RNA splicing and gene expression by eukaryotic riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/17468745/" target="_blank"><b> Nature 447, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Wachter, A. et al. </td>
            <td>Riboswitch control of gene expression in plants by splicing and alternative 3' end processing of mRNAs.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/17993623/" target="_blank"><b> Plant Cell 19, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Thore, S., Frick, C. &amp; Ban, N. </td>
            <td>Structural basis of thiamine pyrophosphate analogues binding to the eukaryotic riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/18533652/" target="_blank"><b> J. Am. Chem. Soc. 130, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Kulshina, N., Edwards, T. E. &amp; Ferré-D’Amaré, A. R. </td>
            <td>Thermodynamic analysis of ligand binding and ligand binding-induced tertiary structure formation by the thiamine pyrophosphate. riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/19948769/" target="_blank"><b> RNA 16, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Wachter, A. </td>
            <td>Riboswitch-mediated control of gene expression in eukaryotes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20009507/" target="_blank"><b> RNA Biol. 7, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Warner, K. D. et al. </td>
            <td>Validating fragment-based drug discovery for biological RNAs: lead fragments bind and remodel the TPP riboswitch specifically.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/24768306/" target="_blank"><b> Chem. Biol. 21, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Yadav, S., Swati, D. &amp; Chandrasekharan, H. </td>
            <td>Thiamine pyrophosphate riboswitch in some representative plant species: a bioinformatics study.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25243980/" target="_blank"><b> J. Comput. Biol. 22, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Mukherjee, S., Retwitzer, Barash, D. &amp; Sengupta, S. </td>
            <td>Phylogenomic and comparative analysis of the distribution and regulatory patterns of TPP riboswitches in fungi.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29615754/" target="_blank"><b> Sci. Rep. 8, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Stav, S. et al. </td>
            <td>Genome-wide discovery of structured noncoding RNAs in bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30902049/" target="_blank"><b> BMC Microbiol. 19, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Subki, A., Ho, C. L., Ismail, N. F. N., Aa, Z. A. &amp; Zn, B. Y. </td>
            <td>Identification and characterisation of thiamine pyrophosphate (TPP) riboswitch in Elaeis guineensis.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/32726320/" target="_blank"><b> PLoS One 15, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Zeller, M. J. et al. </td>
            <td>Subsite Ligand Recognition and Cooperativity in the TPP Riboswitch: Implications for Fragment-Linking in RNA Ligand Discovery.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35060698/" target="_blank"><b> ACS Chem. Biol. 17, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Zeller, M. J. et al. </td>
            <td>SHAPE-enabled fragment-based ligand discovery for RNA.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35561226/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 119, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Lee, H. K. et al. </td>
            <td>Crystal structure of Escherichia coli thiamine pyrophosphate-sensing riboswitch in the apo state.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/TPP" target="_blank"><b> TPP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/37253356/" target="_blank"><b> Structure </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Barrick, J. E. et al. </td>
            <td>New RNA motifs suggest an expanded scope for riboswitches in bacterial genetic control.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Manganese" target="_blank"><b> Manganese</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15096624/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 101, 6421–6426 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Dambach, M. et al. </td>
            <td>The ubiquitous yybP-ykoY riboswitch is a manganese-responsive regulatory element.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Manganese" target="_blank"><b> Manganese</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25794618/" target="_blank"><b> Mol. Cell 57, 1099–1109 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Price, I. R., Gaballa, A., Ding, F., Helmann, J. D. &amp; Ke, A. </td>
            <td>Mn(2+)-sensing mechanisms of yybP-ykoY orphan riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Manganese" target="_blank"><b> Manganese</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25794619/" target="_blank"><b> Mol. Cell 57, 1110–1123 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Bachas, S. T. &amp; Ferré-D’Amaré, A. R. </td>
            <td>Convergent Use of Heptacoordination for Cation Selectivity by RNA and Protein Metalloregulators.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Manganese" target="_blank"><b> Manganese</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29805037/" target="_blank"><b> Cell Chem Biol 25, 962–973.e5 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Martin, J. E. et al. </td>
            <td>A Mn-sensing riboswitch activates expression of a Mn2+/Ca2+ ATPase transporter in Streptococcus.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Manganese" target="_blank"><b> Manganese</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31165873/" target="_blank"><b> Nucleic Acids Res. 47, 6885–6899 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Suddala, K. C. et al. </td>
            <td>Local-to-global signal transduction at the core of a Mn sensing riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Manganese" target="_blank"><b> Manganese</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31541094/" target="_blank"><b> Nat. Commun. 10, 4304 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Mandal, M. et al. </td>
            <td>A glycine-dependent riboswitch that uses cooperative binding to control gene expression</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15472076/" target="_blank"><b> Science 306, 275–279 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Lipfert, J. et al. </td>
            <td>Structural transitions and thermodynamics of a glycine-dependent riboswitch from Vibrio cholerae</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/17118400/" target="_blank"><b> J. Mol. Biol. 365, 1393–1406 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Kwon, M. &amp; Strobel, S. A. </td>
            <td>Chemical basis of glycine riboswitch cooperativity</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/18042658/" target="_blank"><b> RNA 14, 25–34 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Huang, L., Serganov, A. &amp; Patel, D. J. </td>
            <td>Structural insights into ligand recognition by a sensing domain of the cooperative glycine riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21145485/" target="_blank"><b> Mol. Cell 40, 774–786 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Lipfert, J., Sim, A. Y. L., Herschlag, D. &amp; Doniach, S. </td>
            <td>Dissecting electrostatic screening, specific ion binding, and ligand binding in an energetic model for glycine riboswitch folding</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20194520/" target="_blank"><b> RNA 16, 708–719 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Butler, E. B., Xiong, Y., Wang, J. &amp; Strobel, S. A. </td>
            <td>Structural basis of cooperative ligand binding by the glycine riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21439473/" target="_blank"><b> Chem. Biol. 18, 293–298 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Erion, T. V. &amp; Strobel, S. A. </td>
            <td>Identification of a tertiary interaction important for cooperative ligand binding by the glycine riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21098652/" target="_blank"><b> RNA 17, 74–84 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Ruff, K. M. &amp; Strobel, S. A. </td>
            <td>Ligand binding by the tandem glycine riboswitch depends on aptamer dimerization but not double ligand occupancy</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25246650/" target="_blank"><b> RNA 20, 1775–1788 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Ketterer, S., Gladis, L., Kozica, A. &amp; Meier, M. </td>
            <td>Engineering and characterization of fluorogenic glycine riboswitches</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/27220466/" target="_blank"><b> Nucleic Acids Res. 44, 5983–5992 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Ruff, K. M., Muhammad, A., McCown, P. J., Breaker, R. R. &amp; Strobel, S. A. </td>
            <td>Singlet glycine riboswitches bind ligand as well as tandem riboswitches</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/27659053/" target="_blank"><b> RNA 22, 1728–1738 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Babina, A. M., Lea, N. E. &amp; Meyer, M. M. </td>
            <td>In Vivo Behavior of the Tandem Glycine Riboswitch in <i>Bacillus subtilis</i></td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29089431/" target="_blank"><b> MBio 8, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Khani, A., Popp, N., Kreikemeyer, B. &amp; Patenge, N. </td>
            <td>A Glycine Riboswitch in Controls Expression of a Sodium:Alanine Symporter Family Protein Gene</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29527194/" target="_blank"><b> Front. Microbiol. 9, 200 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Zhou, L. et al. </td>
            <td>Characterization and Engineering of a Clostridium Glycine Riboswitch and Its Use To Control a Novel Metabolic Pathway for 5-Aminolevulinic Acid Production in <i>Escherichia coli</i></td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31550137/" target="_blank"><b> ACS Synth. Biol. 8, 2327–2335 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Kappel, K. et al. </td>
            <td>Accelerated cryo-EM-guided determination of three-dimensional RNA-only structures</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/32616928/" target="_blank"><b> Nat. Methods 17, 699–707 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Torgerson, C. D., Hiller, D. A. &amp; Strobel, S. A. </td>
            <td>The asymmetry and cooperativity of tandem glycine riboswitch aptamers</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31992591/" target="_blank"><b> RNA 26, 564–580 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Hong, K.-Q., Zhang, J., Jin, B., Chen, T. &amp; Wang, Z.-W. </td>
            <td>Development and characterization of a glycine biosensor system for fine-tuned metabolic regulation in Escherichia coli</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Glycine" target="_blank"><b> Glycine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35392910/" target="_blank"><b> Microb. Cell Fact. 21, 56 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Barrick, J. E. et al. </td>
            <td>New RNA motifs suggest an expanded scope for riboswitches in bacterial genetic control.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ADP" target="_blank"><b> ADP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15096624/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 101, 6421–6426 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Nelson, J. W., Atilho, R. M., Sherlock, M. E., Stockbridge, R. B. &amp; Breaker, R. R. </td>
            <td>Metabolism of Free Guanidine in Bacteria Is Regulated by a Widespread Riboswitch Class.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ADP" target="_blank"><b> ADP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/27989440/" target="_blank"><b> Mol. Cell 65, 220–230 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sherlock, M. E., Sudarsan, N., Stav, S. &amp; Breaker, R. R. </td>
            <td>Tandem riboswitches form a natural Boolean logic gate to control purine metabolism in bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ADP" target="_blank"><b> ADP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29504937/" target="_blank"><b> Elife 7, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sherlock, M. E., Sadeeshkumar, H. &amp; Breaker, R. R. </td>
            <td>Variant Bacterial Riboswitches Associated with Nucleotide Hydrolase Genes Sense Nucleoside Diphosphates.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/ADP" target="_blank"><b> ADP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30081631/" target="_blank"><b> Biochemistry 58, 401–410 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z. et al. </td>
            <td>Detection of 224 candidate structured RNAs by comparative analysis of specific subsets of intergenic regions.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Li" target="_blank"><b> Li<sup>+</sup></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28977401/" target="_blank"><b> Nucleic Acids Res. 45, 10811–10823 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>White, N., Sadeeshkumar, H., Sun, A., Sudarsan, N. &amp; Breaker, R. R. </td>
            <td>Lithium-sensing riboswitch classes regulate expression of bacterial cation transporter genes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Li" target="_blank"><b> Li<sup>+</sup></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36352003/" target="_blank"><b> Sci. Rep. 12, 19145 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Grundy, F. J. &amp; Henkin, T. M. </td>
            <td>The S box regulon: a new global transcription termination control system for methionine and cysteine biosynthesis genes in gram-positive bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/10094622/" target="_blank"><b> Mol. Microbiol. 30, 737–749 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Winkler, W. C., Nahvi, A., Sudarsan, N., Barrick, J. E., &amp; Breaker, R. R </td>
            <td>An mRNA structure that controls gene expression by binding S-adenosylmethionine.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/12910260/" target="_blank"><b> Nat. Struct. Biol. 10, 701–707 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>McDaniel, B. A., Grundy, F. J., Artsimovitch, I., &amp; Henkin, T. M. </td>
            <td>Transcription termination control of the S box system: direct measurement of S-adenosylmethionine by the leader RNA.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/12626738/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 100, 3083–3088 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Montange, R. K., &amp; Batey, R. T. </td>
            <td>Structure of the S-adenosylmethionine riboswitch regulatory mRNA element.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16810258/" target="_blank"><b> Nature 441, 1172–1175 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z., Barrick, J. E., Yao, Z., Roth, A., Kim, J. N., Gore, J., Wang, J. X., Lee, E. R., Block, K. F., Sudarsan, N., Neph, S., Tompa, M., Ruzzo, W. L., &amp; Breaker, R. R. </td>
            <td>Identification of 22 candidate structured RNAs in bacteria using the CMfinder comparative genomics pipeline.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/17621584/" target="_blank"><b> Nucleic Acids Res. 35, 4809–4819 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Yao, Z., Barrick, J., Weinberg, Z., Neph, S., Breaker, R., Tompa, M., and Ruzzo, W.L. </td>
            <td>A computational pipeline for high-throughput discovery of cis-regulatory noncoding RNA in prokaryotes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/17616982/" target="_blank"><b> PLoS Comput. Biol. 3, e126 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z., Regulski, E. E., Hammond, M. C., Barrick, J. E., Yao, Z., Ruzzo, W. L., &amp; Breaker, R. R. </td>
            <td>The aptamer core of SAM-IV riboswitches mimics the ligand-binding site of SAM-I riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/18369181/" target="_blank"><b> RNA 14, 822–828 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Montange, R. K., Mondragón, E., van Tyne, D., Garst, A. D., Ceres, P., &amp; Batey, R. T. </td>
            <td>Discrimination between closely related cellular metabolites by the SAM-I riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20006621/" target="_blank"><b> J. Mol. Biol. 396, 761–772 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z., Wang, J. X., Bogue, J., Yang, J., Corbino, K., Moy, R. H., &amp; Breaker, R. R. </td>
            <td>Comparative genomics reveals 104 candidate structured RNAs from bacteria, archaea, and their metagenomes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20230605/" target="_blank"><b> Genome Biol. 11, R31 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Stoddard, C. D., Montange, R. K., Hennelly, S. P., Rambo, R. P., Sanbonmatsu, K. Y., &amp; Batey, R. T. </td>
            <td>Free state conformational sampling of the SAM-I riboswitch aptamer domain.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20637415/" target="_blank"><b> Structure 18, 787–797 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Lu, C., Ding, F., Chowdhury, A., Pradhan, V., Tomsic, J., Holmes, W. M., Henkin, T. M., &amp; Ke, A. </td>
            <td>SAM recognition and conformational switching mechanism in the Bacillus subtilis yitJ S box/SAM-I riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20951706/" target="_blank"><b> J. Mol. Biol. 404, 803–818 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Heppell, B., Blouin, S., Dussault, A. M., Mulhbacher, J., Ennifar, E., Penedo, J. C., &amp; Lafontaine, D. A. </td>
            <td>Molecular insights into the ligand-controlled organization of the SAM-I riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21532599/" target="_blank"><b> Nat. Chem. Biol. 7, 384–392 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Eschbach, S. H., St-Pierre, P., Penedo, J. C., &amp; Lafontaine, D. A. </td>
            <td>Folding of the SAM-I riboswitch: a tale with a twist.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/22336759/" target="_blank"><b> RNA Biol. 9, 535–541 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Boyapati, V. K., Huang, W., Spedale, J., &amp; Aboul-Ela, F. </td>
            <td>Basis for ligand discrimination between ON and OFF state riboswitch conformations: the case of the SAM-I riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/22543867/" target="_blank"><b> RNA 18, 1230–1243 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Trausch, J. J., Xu, Z., Edwards, A. L., Reyes, F. E., Ross, P. E., Knight, R., &amp; Batey, R. T. </td>
            <td>Structural basis for diversity in the SAM clan of riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/24753586/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 111, 6624–6629 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Price IR, Grigg JC, Ke A. </td>
            <td>Common themes and differences in SAM recognition among SAM riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/24863160/" target="_blank"><b> Biochim Biophys Acta.1839</b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Dussault, A.-M., Dubé, A., Jacques, F., Grondin, J. P., and Lafontaine, D. A. </td>
            <td>Ligand recognition and helical stacking formation are intimately linked in the SAM-I riboswitch regulatory mechanism.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28701520/" target="_blank"><b> RNA 23, 1539–1551 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Manz, C., Kobitski, A. Y., Samanta, A., Keller, B. G., Jäschke, A., &amp; Nienhaus, G. U. </td>
            <td>Single-molecule FRET reveals the energy landscape of the full-length SAM-I riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28920931/" target="_blank"><b> Nat. Chem. Biol. 13, 1172–1178 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Manz, C., Kobitski, A. Y., Samanta, A., Jäschke, A., &amp; Nienhaus, G. U. </td>
            <td>The multi-state energy landscape of the SAM-I riboswitch: A single-molecule Förster resonance energy transfer spectroscopy study.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29604896/" target="_blank"><b> J. Chem. Phys. 148, 123324 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Zhang, K., Li, S., Kappel, K., Pintilie, G., Su, Z., Mou, T. C., Schmid, M. F., Das, R., &amp; Chiu, W. </td>
            <td>Cryo-EM structure of a 40 kDa SAM-IV riboswitch RNA at 3.7 Å resolution.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31796736/" target="_blank"><b> Nat. Commun. 10, 5511 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Tang, D. J., Du, X., Shi, Q., Zhang, J. L., He, Y. P., Chen, Y. M., Ming, Z., Wang, D., Zhong, W. Y., Liang, Y. W., Liu, J. Y., Huang, J. M., Zhong, Y. S., An, S. Q., Gu, H., &amp; Tang, J. L. </td>
            <td>A SAM-I riboswitch with the ability to sense and respond to uncharged initiator tRNA.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/32493973/" target="_blank"><b> Nat. Commun. 11, 2794 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Traykovska, M., &amp; Penchovsky, R. </td>
            <td>Targeting SAM-I riboswitch using antisense oligonucleotide technology for inhibiting the growth of staphylococcus aureus and listeria monocytogenes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36421306/" target="_blank"><b> Antibiotics </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Zheng L, Song Q, Xu X, Shen X, Li C, Li H, Chen H, Ren A. </td>
            <td>Structure-based insights into recognition and regulation of SAM-sensing riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-I_clan" target="_blank"><b> SAM-I_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36459353/" target="_blank"><b> Sci China Life Sci.66</b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Ames, T. D., Rodionov, D. A., Weinberg, Z. &amp; Breaker, R. R. </td>
            <td>A eubacterial riboswitch class that senses the coenzyme tetrahydrofolate.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/THF" target="_blank"><b> THF</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20659680/" target="_blank"><b> Chem. Biol. 17, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Huang, L., Ishibe-Murakami, S., Patel, D. J. &amp; Serganov, A. </td>
            <td>Long-range pseudoknot interactions dictate the regulatory response in the tetrahydrofolate riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/THF" target="_blank"><b> THF</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21873197/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 108, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Trausch, J. J., Ceres, P., Reyes, F. E. &amp; Batey, R. T. </td>
            <td>The structure of a tetrahydrofolate-sensing riboswitch reveals two ligand binding sites in a single aptamer.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/THF" target="_blank"><b> THF</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21906956/" target="_blank"><b> Structure 19, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Trausch, J. J. &amp; Batey, R. T. </td>
            <td>A disconnect between high-affinity binding and efficient regulation by antifolates and purines in the tetrahydrofolate riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/THF" target="_blank"><b> THF</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/24388757/" target="_blank"><b> Chem. Biol. 21, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z. et al. </td>
            <td>Detection of 224 candidate structured RNAs by comparative analysis of specific subsets of intergenic regions.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/THF" target="_blank"><b> THF</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28977401/" target="_blank"><b> Nucleic Acids Res. 45, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Chen, X., Mirihana, A. G. &amp; Breaker, R. R. </td>
            <td>Biochemical validation of a second class of tetrahydrofolate riboswitches in bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/THF" target="_blank"><b> THF</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31186369/" target="_blank"><b> RNA 25, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Wilt, H. M., Yu, P., Tan, K., Wang, Y. X. &amp; Stagno, J. R. </td>
            <td>Tying the knot in the tetrahydrofolate (THF) riboswitch: A molecular basis for gene regulation.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/THF" target="_blank"><b> THF</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/33571639/" target="_blank"><b> J. Struct. Biol. 213, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Xu, L., Xiao, Y., Zhang, J. &amp; Fang, X. </td>
            <td>Structural insights into translation regulation by the THF-II riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/THF" target="_blank"><b> THF</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36620887/" target="_blank"><b> Nucleic Acids Res. 51, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Barrick, J. E. et al. </td>
            <td>New RNA motifs suggest an expanded scope for riboswitches in bacterial genetic control.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-AMP" target="_blank"><b> c-di-AMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/15096624/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 101, 6421–6426 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Watson, P. Y. &amp; Fedor, M. J. </td>
            <td>The ydaO motif is an ATP-sensing riboswitch in Bacillus subtilis.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-AMP" target="_blank"><b> c-di-AMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/23086297/" target="_blank"><b> Nat. Chem. Biol. 8, 963–965 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Nelson, J. W. et al. </td>
            <td>Riboswitches in eubacteria sense the second messenger c-di-AMP.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-AMP" target="_blank"><b> c-di-AMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/24141192/" target="_blank"><b> Nat. Chem. Biol. 9, 834–839 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Ren, A. &amp; Patel, D. J. </td>
            <td>c-di-AMP binds the ydaO riboswitch in two pseudo-symmetry-related pockets.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-AMP" target="_blank"><b> c-di-AMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25086509/" target="_blank"><b> Nat. Chem. Biol. 10, 780–786 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Gao, A. &amp; Serganov, A. </td>
            <td>Structural insights into recognition of c-di-AMP by the ydaO riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-AMP" target="_blank"><b> c-di-AMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25086507/" target="_blank"><b> Nat. Chem. Biol. 10, 787–792 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Jones, C. P. &amp; Ferré-D’Amaré, A. R. </td>
            <td>Crystal structure of a c-di-AMP riboswitch reveals an internally pseudo-dimeric RNA.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-AMP" target="_blank"><b> c-di-AMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25271255/" target="_blank"><b> EMBO J. 33, 2692–2703 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Wang, X. et al. </td>
            <td>A c-di-AMP riboswitch controlling kdpFABC operon transcription regulates the potassium transporter system in Bacillus thuringiensis.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-AMP" target="_blank"><b> c-di-AMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31044176/" target="_blank"><b> Commun Biol 2, 151 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Reich, S. J. et al. </td>
            <td>C-di-AMP Is a Second Messenger in Corynebacterium glutamicum That Regulates Expression of a Cell Wall-Related Peptidase via a Riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-AMP" target="_blank"><b> c-di-AMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36838266/" target="_blank"><b> Microorganisms 11, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z. et al. </td>
            <td>Detection of 224 candidate structured RNAs by comparative analysis of specific subsets of intergenic regions</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/NAD" target="_blank"><b> NAD<sup>+</sup>-I</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28977401/" target="_blank"><b> Nucleic Acids Res. 45, 10811–10823 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Malkowski, S. N., Spencer, T. C. J. &amp; Breaker, R. R. </td>
            <td>Evidence that the nadA motif is a bacterial riboswitch for the ubiquitous enzyme  cofactor NAD</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/NAD" target="_blank"><b> NAD<sup>+</sup>-I</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/31467147/" target="_blank"><b> RNA 25, 1616–1627 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Huang, L., Wang, J. &amp; Lilley, D. M. J. </td>
            <td>Structure and ligand binding of the ADP-binding domain of the NAD<sup>+</sup> riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/NAD" target="_blank"><b> NAD<sup>+</sup>-I</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/32295864/" target="_blank"><b> RNA 26, 878–887 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Chen, H. et al. </td>
            <td>Structural distinctions between NAD<sup>+</sup> riboswitch domains 1 and 2 determine  differential folding and ligand binding</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/NAD" target="_blank"><b> NAD<sup>+</sup>-I</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/33170270/" target="_blank"><b> Nucleic Acids Res. 48, 12394–12406 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Brewer, K. I. et al. </td>
            <td>Comprehensive discovery of novel structured noncoding RNAs in 26 bacterial genomes</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/NAD2" target="_blank"><b> NAD<sup>+</sup>-II</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/33970790/" target="_blank"><b> RNA Biol. 18, 2417–2432 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Panchapakesan, S. S. S., Corey, L., Malkowski, S. N., Higgs, G. &amp; Breaker, R. R. </td>
            <td>A second riboswitch class for the enzyme cofactor NAD</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/NAD2" target="_blank"><b> NAD<sup>+</sup>-II</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/33087526/" target="_blank"><b> RNA 27, 99–105 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Xu, X. et al. </td>
            <td>Structure-based investigations of the NAD<sup>+</sup>-II riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/NAD2" target="_blank"><b> NAD<sup>+</sup>-II</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36610789/" target="_blank"><b> Nucleic Acids Res. 51, 54–67 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Peng, X., Liao, W., Lin, X., Lilley, D. M. J. &amp; Huang, L. </td>
            <td>Crystal structures of the NAD<sup>+</sup>-II riboswitch reveal two distinct ligand-binding pockets</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/NAD2" target="_blank"><b> NAD<sup>+</sup>-II</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36840714/" target="_blank"><b> Nucleic Acids Res. 51, 2904–2914 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Nou, X. &amp; Kadner, R. J. </td>
            <td>Adenosylcobalamin inhibits ribosome binding to btuB RNA.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Cobalamine" target="_blank"><b> Cobalamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/10852957/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 97, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Nahvi, A. et al. </td>
            <td>Genetic control by a metabolite binding mRNA.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Cobalamine" target="_blank"><b> Cobalamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/12323379/" target="_blank"><b> Chem. Biol. 9, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Vitreschak, A. G., Rodionov, D. A., Mironov, A. A. &amp; Gelfand, M. S. </td>
            <td>Regulation of the vitamin B12 metabolism and transport in bacteria by a conserved RNA structural element.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Cobalamine" target="_blank"><b> Cobalamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/12923257/" target="_blank"><b> RNA 9, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Rodionov, D. A., Vitreschak, A. G., Mironov, A. A. &amp; Gelfand, M. S. </td>
            <td>Comparative genomics of the vitamin B12 metabolism and regulation in prokaryotes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Cobalamine" target="_blank"><b> Cobalamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/12869542/" target="_blank"><b> J. Biol. Chem. 278, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Nahvi, A., Barrick, J. E. &amp; Breaker, R. R. </td>
            <td>Coenzyme B12 riboswitches are widespread genetic control elements in prokaryotes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Cobalamine" target="_blank"><b> Cobalamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/14704351/" target="_blank"><b> Nucleic Acids Res. 32, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z. et al. </td>
            <td>Comparative genomics reveals 104 candidate structured RNAs from bacteria, archaea, and their metagenomes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Cobalamine" target="_blank"><b> Cobalamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20230605/" target="_blank"><b> Genome Biol. 11, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Johnson, J. E., Reyes, F. E., Polaski, J. T. &amp; Batey, R. T. </td>
            <td>B12 cofactors directly stabilize an mRNA regulatory switch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Cobalamine" target="_blank"><b> Cobalamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/23064232/" target="_blank"><b> Nature 492, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Peselis, A. &amp; Serganov, A. </td>
            <td>Structural insights into ligand binding and gene expression control by an adenosylcobalamin riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Cobalamine" target="_blank"><b> Cobalamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/23064646/" target="_blank"><b> Nat. Struct. Mol. Biol. 19, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Choudhary, P. K. &amp; Sigel, R. K. </td>
            <td>Mg<sup>2+</sup>-induced conformational changes in the btuB riboswitch from <i>E. coli</i>.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Cobalamine" target="_blank"><b> Cobalamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/24243114/" target="_blank"><b> RNA 20, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Choudhary, P. K., Gallo, S. &amp; Sigel, R. K. </td>
            <td>Tb(3+)-Cleavage Assays Reveal Specific Mg(2+) Binding Sites Necessary to Pre-fold the btuB Riboswitch for AdoCbl Binding.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Cobalamine" target="_blank"><b> Cobalamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28377919/" target="_blank"><b> Frontiers in chemistry 5, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Polaski, J. T., Webster, S. M., Johnson, J. E. &amp; Batey, R. T. </td>
            <td>Cobalamin riboswitches exhibit a broad range of ability to discriminate between methylcobalamin and adenosylcobalamin.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Cobalamine" target="_blank"><b> Cobalamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28483920/" target="_blank"><b> J. Biol. Chem. 292, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Chan, C. W. &amp; Mondragón, A. </td>
            <td>Crystal structure of an atypical cobalamin riboswitch reveals RNA structural adaptability as basis for promiscuous ligand binding.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Cobalamine" target="_blank"><b> Cobalamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/32544228/" target="_blank"><b> Nucleic Acids Res. 48, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Ma, B., Bai, G., Nussinov, R., Ding, J. &amp; Wang, Y.-X. </td>
            <td>Conformational Ensemble of AdoCbl Riboswitch Provides Stable Structural Elements for Conformation Selection and Population Shift in Cobalamin Recognition.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Cobalamine" target="_blank"><b> Cobalamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/33683130/" target="_blank"><b> J. Phys. Chem. B 125, 2589–2596 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Lennon, S. R. et al. </td>
            <td>Targeting Riboswitches with Beta-Axial-Substituted Cobalamins.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Cobalamine" target="_blank"><b> Cobalamine</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/37094176/" target="_blank"><b> ACS Chem. Biol. 18, 1136–1147 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Corbino, K. A., Barrick, J. E., Lim, J., Welz, R., Tucker, B. J., Puskarz, I., Mandal, M., Rudnick, N. D., &amp; Breaker, R. R. </td>
            <td>Evidence for a second class of S-adenosylmethionine riboswitches and other regulatory RNA motifs in alpha-proteobacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-II_clan" target="_blank"><b> SAM-II_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16086852/" target="_blank"><b> Genome Biol. 6, R70 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Lim, J., Winkler, W. C., Nakamura, S., Scott, V., &amp; Breaker, R. R. </td>
            <td>Molecular-recognition characteristics of SAM-binding riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-II_clan" target="_blank"><b> SAM-II_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16381055/" target="_blank"><b> Angew. Chem. Int. Ed Engl. 45, 964–968 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Gilbert, S. D., Rambo, R. P., Van Tyne, D., &amp; Batey, R. T. </td>
            <td>Structure of the SAM-II riboswitch bound to S-adenosylmethionine.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-II_clan" target="_blank"><b> SAM-II_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/18204466/" target="_blank"><b> Nat. Struct. Mol. Biol. 15, 177–182 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Meyer, M. M., Ames, T. D., Smith, D. P., Weinberg, Z., Schwalbach, M. S., Giovannoni, S. J., &amp; Breaker, R. R. </td>
            <td>Identification of candidate structured RNAs in the marine organism 'Candidatus Pelagibacter Ubique'.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-II_clan" target="_blank"><b> SAM-II_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/19531245/" target="_blank"><b> BMC Genomics 10, 268 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Poiata, E., Meyer, M. M., Ames, T. D., &amp; Breaker, R. R. </td>
            <td>A variant riboswitch aptamer class for S-adenosylmethionine common in marine bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-II_clan" target="_blank"><b> SAM-II_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/19776155/" target="_blank"><b> RNA 15, 2046–2056 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Haller, A., Rieder, U., Aigner, M., Blanchard, S. C., &amp; Micura, R. </td>
            <td>Conformational capture of the SAM-II riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-II_clan" target="_blank"><b> SAM-II_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21532598/" target="_blank"><b> Nat. Chem. Biol. 7, 393–400 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Chen, B., Zuo, X., Wang, Y. X., &amp; Dayie, T. K. </td>
            <td>Multiple conformations of SAM-II riboswitch detected with SAXS and NMR spectroscopy.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-II_clan" target="_blank"><b> SAM-II_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/22139931/" target="_blank"><b> Nucleic Acids Res. 40, 3117–3130 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Doshi, U., Kelley, J. M., &amp; Hamelberg, D. </td>
            <td>Atomic-level insights into metabolite recognition and specificity of the SAM-II riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-II_clan" target="_blank"><b> SAM-II_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/22194311/" target="_blank"><b> RNA 18, 300–307 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Price IR, Grigg JC, Ke A. </td>
            <td>Common themes and differences in SAM recognition among SAM riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-II_clan" target="_blank"><b> SAM-II_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/24863160/" target="_blank"><b> Biochim Biophys Acta.1839</b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Xue, X., Yongjun, W., &amp; Zhihong, L. </td>
            <td>Folding of SAM-II riboswitch explored by replica-exchange molecular dynamics simulation.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-II_clan" target="_blank"><b> SAM-II_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25451761/" target="_blank"><b> J. Theor. Biol. 365, 265–269 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Chen, B., LeBlanc, R., &amp; Dayie, T. K. </td>
            <td>SAM-II riboswitch samples at least two conformations in solution in the absence of ligand: implications for recognition.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-II_clan" target="_blank"><b> SAM-II_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/26800479/" target="_blank"><b> Angew. Chem. Int. Ed Engl. 55, 2724–2727 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Roy, S., Lammert, H., Hayes, R. L., Chen, B., LeBlanc, R., Dayie, T. K., Onuchic, J. N., &amp; Sanbonmatsu, K. Y. </td>
            <td>A magnesium-induced triplex pre-organizes the SAM-II riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-II_clan" target="_blank"><b> SAM-II_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28248966/" target="_blank"><b> PLoS Comput. Biol. 13, e1005406 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Huang, L., &amp; Lilley, D. M. J. </td>
            <td>Structure and ligand binding of the SAM-V riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-II_clan" target="_blank"><b> SAM-II_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29931337/" target="_blank"><b> Nucleic Acids Res. 46, 6869–6879 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Zheng L, Song Q, Xu X, Shen X, Li C, Li H, Chen H, Ren A. </td>
            <td>Structure-based insights into recognition and regulation of SAM-sensing riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-II_clan" target="_blank"><b> SAM-II_clan</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36459353/" target="_blank"><b> Sci China Life Sci.66</b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Roth, A. et al </td>
            <td>A riboswitch selective for the queuosine precursor preQ<sub>1</sub> contains an unusually small aptamer domain</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/17384645/" target="_blank"><b> Nat. Struct. Mol. Biol. 14, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Meyer, M. M., Roth, A., Chervin, S. M., Garcia, G. A. &amp; Breaker, R. R </td>
            <td>Confirmation of a second natural preQ<sub>1</sub> aptamer class in Streptococcaceae bacteria</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/18305186/" target="_blank"><b> RNA 14, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Klein, D. J., Edwards, T. E. &amp; Ferré-D’Amaré, A. R. </td>
            <td>Cocrystal structure of a class I preQ<sub>1</sub> riboswitch reveals a pseudoknot recognizing an essential hypermodified nucleobase.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/19234468/" target="_blank"><b> Nat. Struct. Mol. Biol. 16, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Kang, M., Peterson, R. &amp; Feigon, J. </td>
            <td>Structural Insights into riboswitch control of the biosynthesis of queuosine, a  modified nucleotide found in the anticodon of Trna.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/19285444/" target="_blank"><b> Mol. Cell 33, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Spitale, R. C., Torelli, A. T., Krucinska, J., Bandarian, V. &amp; Wedekind, J. E. </td>
            <td>The Structural Basis for Recognition of the PreQ<sub>0</sub> Metabolite by an Unusually Small Riboswitch Aptamer Domain.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/19261617/" target="_blank"><b> J. Biol. Chem. 284, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Jenkins, J. L., Krucinska, J., McCarty, R. M., Bandarian, V. &amp; Wedekind, J. E. </td>
            <td>Comparison of a PreQ<sub>1</sub> Riboswitch Aptamer in Metabolite-bound and Free States with Implications for Gene Regulation.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21592962/" target="_blank"><b> J. Biol. Chem. 286, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Rieder, U., Kreutz, C. &amp; Micura, R. </td>
            <td>Folding of a transcriptionally acting preQ1 riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20534493/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 107, 10804–10809 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Santner, T., Rieder, U., Kreutz, C. &amp; Micura, R. </td>
            <td>Pseudoknot preorganization of the preQ1 class I riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/22775200/" target="_blank"><b> J. Am. Chem. Soc. 134, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Soulière, M. F. et al. </td>
            <td>Tuning a riboswitch response through structural extension of a pseudoknot.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/23940363/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 110, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Liberman, J. A., Salim, M., Krucinska, J. &amp; Wedekind, J. E. </td>
            <td>Structure of a class II preQ<sub>1</sub> riboswitch reveals ligand recognition by a new fold.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/23584677/" target="_blank"><b> Nat. Chem. Biol. 9, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Kang, M., Eichhorn, C. D. &amp; Feigon, J. </td>
            <td>Structural determinants for ligand capture by a class II preQ<sub>1</sub> riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/24469808/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 111, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>McCown, P. J., Liang, J. J., Weinberg, Z. &amp; Breaker, R. R. </td>
            <td>Structural, Functional, and Taxonomic Diversity of Three PreQ<sub>1</sub> Riboswitch Classes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25036777/" target="_blank"><b> Chem. Biol. 21, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Liberman, J. A. </td>
            <td>Structural analysis of a class III preQ<sub>1</sub> riboswitch reveals an aptamer distant from a ribosome-binding site regulated by fast dynamics.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/26106162/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 112, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Widom, J. R. et al. </td>
            <td>Ligand Modulates Cross-Coupling between Riboswitch Folding and Transcriptional Pausing.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30388413/" target="_blank"><b> Mol. Cell 72, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Connelly, C. M. et al. </td>
            <td>Synthetic ligands for PreQ<sub>1</sub> riboswitches provide structural and mechanistic insights into targeting RNA tertiary structure.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30940810/" target="_blank"><b> Nat. Commun. 10, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Schroeder, G. M. et al. </td>
            <td>Analysis of a preQ<sub>1</sub>-I riboswitch in effector-free and bound states reveals a metabolite-programmed nucleobase-stacking spine that controls gene regulation.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/32597951/" target="_blank"><b> Nucleic Acids Res. 48, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Flemmich, L., Heel, S., Moreno, S., Breuker, K. &amp; Micura, R. </td>
            <td>A natural riboswitch scaffold with self-methylation activity.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/34162884/" target="_blank"><b> Nat. Commun. 12, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Schroeder, G. M. et al. </td>
            <td>A small RNA that cooperatively senses two stacked metabolites in one pocket for gene control.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/PreQ" target="_blank"><b> PreQ<sub>1</sub></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35017488/" target="_blank"><b> Nat. Commun. 13, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z. et al. </td>
            <td>Comparative genomics reveals 104 candidate structured RNAs from bacteria, archaea, and their metagenomes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Fluoride" target="_blank"><b> Fluoride</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20230605/" target="_blank"><b> Genome Biol. 11, R31 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Baker, J. L. et al. </td>
            <td>Widespread genetic switches and toxicity resistance proteins for fluoride.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Fluoride" target="_blank"><b> Fluoride</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/22194412/" target="_blank"><b> Science 335, 233–235 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Ren, A., Rajashankar, K. R. &amp; Patel, D. J. </td>
            <td>Fluoride ion encapsulation by Mg2+ ions and phosphates in a fluoride riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Fluoride" target="_blank"><b> Fluoride</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/22678284/" target="_blank"><b> Nature 486, 85–89 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Chawla, M., Credendino, R., Poater, A., Oliva, R. &amp; Cavallo, L. </td>
            <td>Structural stability, acidity, and halide selectivity of the fluoride riboswitch recognition site.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Fluoride" target="_blank"><b> Fluoride</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25487435/" target="_blank"><b> J. Am. Chem. Soc. 137, 299–306 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Zhao, B., Guffy, S. L., Williams, B. &amp; Zhang, Q. </td>
            <td>An excited state underlies gene regulation of a transcriptional riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Fluoride" target="_blank"><b> Fluoride</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28719589/" target="_blank"><b> Nat. Chem. Biol. 13, 968–974 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Chauvier, A., Ajmera, P., Yadav, R. &amp; Walter, N. G. </td>
            <td>Dynamic competition between a ligand and transcription factor NusA governs riboswitch-mediated transcription regulation.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Fluoride" target="_blank"><b> Fluoride</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/34782462/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 118, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Yadav, R., Widom, J. R., Chauvier, A. &amp; Walter, N. G. </td>
            <td>An anionic ligand snap-locks a long-range interaction in a magnesium-folded riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Fluoride" target="_blank"><b> Fluoride</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35017489/" target="_blank"><b> Nat. Commun. 13, 207 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Fuchs, R. T., Grundy, F. J., &amp; Henkin, T. M. </td>
            <td>The S(MK) box is a new SAM-binding RNA for translational regulation of SAM synthetase.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-III" target="_blank"><b> SAM-III</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/16491091/" target="_blank"><b> Nat. Struct. Mol. Biol. 13, 226–233 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Fuchs, R. T., Grundy, F. J., &amp; Henkin, T. M. </td>
            <td>S-adenosylmethionine directly inhibits binding of 30S ribosomal subunits to the S(MK) box translational riboswitch RNA.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-III" target="_blank"><b> SAM-III</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/17360376/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 104, 4876–4880 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Lu, C., Smith, A. M., Fuchs, R. T., Ding, F., Rajashankar, K., Henkin, T. M., &amp; Ke, A. </td>
            <td>Crystal structures of the SAM-III/S(MK) riboswitch reveal the SAM-dependent translation inhibition mechanism.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-III" target="_blank"><b> SAM-III</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/18806797/" target="_blank"><b> Nat. Struct. Mol. Biol. 15, 1076–1083 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Priyakumar U. D. </td>
            <td>Atomistic details of the ligand discrimination mechanism of S(MK)/SAM-III riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-III" target="_blank"><b> SAM-III</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20614931/" target="_blank"><b> J. Phys. Chem. B 114, 9920–9925 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Smith, A. M., Fuchs, R. T., Grundy, F. J., &amp; Henkin, T. M. </td>
            <td>The SAM-responsive S(MK) box is a reversible riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-III" target="_blank"><b> SAM-III</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21143313/" target="_blank"><b> Mol. Microbiol. 78, 1393–1402 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Lu, C., Smith, A. M., Ding, F., Chowdhury, A., Henkin, T. M., &amp; Ke, A. </td>
            <td>Variable sequences outside the SAM-binding core critically influence the conformational dynamics of the SAM-III/SMK box riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-III" target="_blank"><b> SAM-III</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21549712/" target="_blank"><b> J. Mol. Biol. 409, 786–799 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Price IR, Grigg JC, Ke A. </td>
            <td>Common themes and differences in SAM recognition among SAM riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-III" target="_blank"><b> SAM-III</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/24863160/" target="_blank"><b> Biochim Biophys Acta.1839</b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Suresh, G., Srinivasan, H., Nanda, S., &amp; Priyakumar, U. D. </td>
            <td>Ligand-induced stabilization of a duplex-like architecture is crucial for the switching mechanism of the SAM-III riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-III" target="_blank"><b> SAM-III</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/27249101/" target="_blank"><b> Biochemistry 55, 3349–3360 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Gong, S., Wang, Y., Wang, Z., Wang, Y., &amp; Zhang, W. </td>
            <td>Reversible-switch mechanism of the SAM-III riboswitch.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-III" target="_blank"><b> SAM-III</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/27934232/" target="_blank"><b> J. Phys. Chem. B 120, 12305–12311 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Chen, J., Zeng, Q., Wang, W., Sun, H., &amp; Hu, G. </td>
            <td>Decoding the identification mechanism of an SAM-III riboswitch on ligands through multiple independent gaussian-accelerated molecular dynamics simulations.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-III" target="_blank"><b> SAM-III</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36440874/" target="_blank"><b> J. Chem. Inf. Model. 62, 6118–6132 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Zheng L, Song Q, Xu X, Shen X, Li C, Li H, Chen H, Ren A. </td>
            <td>Structure-based insights into recognition and regulation of SAM-sensing riboswitches.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/SAM-III" target="_blank"><b> SAM-III</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/36459353/" target="_blank"><b> Sci China Life Sci.66</b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Begley, T. P. et al. </td>
            <td>Thiamin biosynthesis in prokaryotes.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/HMP-PP" target="_blank"><b> HMP-PP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/10382260/" target="_blank"><b> Arch. Microbiol. 171, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Stav, S. et al. </td>
            <td>Genome-wide discovery of structured noncoding RNAs in bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/HMP-PP" target="_blank"><b> HMP-PP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30902049/" target="_blank"><b> BMC Microbiol. 19, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Atilho, R. M., Mirihana, A. G., Greenlee, E. B., Knecht, K. M. &amp; Breaker, R. R. </td>
            <td>A bacterial riboswitch class for the thiamin precursor HMP-PP employs a terminator-embedded aptamer.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/HMP-PP" target="_blank"><b> HMP-PP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/30950790/" target="_blank"><b> Elife 8, </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z. et al. </td>
            <td>Detection of 224 candidate structured RNAs by comparative analysis of specific subsets of intergenic regions.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Na" target="_blank"><b> Na<sup>+</sup></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/28977401/" target="_blank"><b> Nucleic Acids Res. 45, 10811–10823 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>White, N., Sadeeshkumar, H., Sun, A., Sudarsan, N. &amp; Breaker, R. R. </td>
            <td>Na riboswitches regulate genes for diverse physiological processes in bacteria.</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/Na" target="_blank"><b> Na<sup>+</sup></b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35879547/" target="_blank"><b> Nat. Chem. Biol. 18, 878–885 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Weinberg, Z. et al. </td>
            <td>Identification of 22 candidate structured RNAs in bacteria using the CMfinder comparative genomics pipeline</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-GMP" target="_blank"><b> c-di-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/17621584/" target="_blank"><b> Nucleic Acids Res. 35, 4809–4819 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Sudarsan, N. et al. </td>
            <td>Riboswitches in eubacteria sense the second messenger cyclic di-GMP</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-GMP" target="_blank"><b> c-di-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/18635805/" target="_blank"><b> Science 321, 411–413 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Smith, K. D. et al </td>
            <td>Structural basis of ligand binding by a c-di-GMP riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-GMP" target="_blank"><b> c-di-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/19898477/" target="_blank"><b> Nat. Struct. Mol. Biol. 16, 1218–1223 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Smith, K. D., Lipchock, S. V., Livingston, A. L., Shanahan, C. A. &amp; Strobel, S. A. </td>
            <td>Structural and biochemical determinants of ligand binding by the c-di-GMP riboswitch</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-GMP" target="_blank"><b> c-di-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20690679/" target="_blank"><b> Biochemistry 49, 7351–7359 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Lee, E. R., Baker, J. L., Weinberg, Z., Sudarsan, N. &amp; Breaker, R. R. </td>
            <td>An allosteric self-splicing ribozyme triggered by a bacterial second messenger</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-GMP" target="_blank"><b> c-di-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/20705859/" target="_blank"><b> Science 329, 845–848 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Smith, K. D., Shanahan, C. A., Moore, E. L., Simon, A. C. &amp; Strobel, S. A. </td>
            <td>Structural basis of differential ligand recognition by two classes of bis-(3'-5')-cyclic dimeric guanosine monophosphate-binding riboswitches</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-GMP" target="_blank"><b> c-di-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/21518891/" target="_blank"><b> Proc. Natl. Acad. Sci. U. S. A. 108, 7757–7762 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Smith, K. D., Lipchock, S. V. &amp; Strobel, S. A. </td>
            <td>Structural and biochemical characterization of linear dinucleotide analogues bound to the c-di-GMP-I aptamer</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-GMP" target="_blank"><b> c-di-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/22148472/" target="_blank"><b> Biochemistry 51, 425–432 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Kalia, D. et al. </td>
            <td>Nucleotide, c-di-GMP, c-di-AMP, cGMP, cAMP, (p)ppGpp signaling in bacteria and implications in pathogenesis</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-GMP" target="_blank"><b> c-di-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/23023210/" target="_blank"><b> Chem. Soc. Rev. 42, 305–341 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Bordeleau, E. et al. </td>
            <td>Cyclic di-GMP riboswitch-regulated type IV pili contribute to aggregation of Clostridium difficile</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-GMP" target="_blank"><b> c-di-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/25512308/" target="_blank"><b> J. Bacteriol. 197, 819–832 </b></a></td>
        </tr>
                    
        <tr>
            <td>updating</td>
            <td>Inuzuka, S. et al. </td>
            <td>Recognition of cyclic-di-GMP by a riboswitch conducts translational repression through masking the ribosome-binding site distant from the aptamer domain</td>
            <td><a href="https://riboswitch.ribocentre.org/docs/c-di-GMP" target="_blank"><b> c-di-GMP</b></a></td>
            <td>updating</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/29693296/" target="_blank"><b> Genes Cells 23, 435–447 </b></a></td>
        </tr>
                    
</table><br /><br />
        
</div></body></html>

    
    </section>
    
  </div>
  <br>
</div>
<br>
        </div>
      </div>
    <div id="footer" class="panel">
  <div class="panel-footer">
	<div class="container-fluid2">
	  <div class="row">
		<div class="col-sm-4">
			
		  <p>&copy 2023 Ribocentre. Site made with <a href="https://jekyllrb.com">Jekyll</a>;  <a href="https://www.ribocentre.org/aboutwebsite.html">copy and  modify it for your own research group.</a></p>
              <p>We are part of the <a href="https://gzlab.ac.cn">Guangzhou Laboratory</a> and <a href="http://www.gzsys.org.cn/">Sun Yat-sen Memorial Hospital of Sun Yat-sen University</a>.</p>
            

		   <p>  </p><p>
            		  
            
		</div>
		<div class="col-sm-4">
		  Funding:<br />
		  - <a href="https://www.nsfc.gov.cn/">NSFC</a> <br />
		  - <a href="http://www.most.gov.cn/">MOST IT+BT project</a> <br />
          - <a href="https://gzlab.ac.cn">GZlab starting grant</a><br />
		  <br>
		  <script async src="//busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
		  <!--<span class="glyphicon glyphicon-user"></span>&nbsp;&nbsp;<span id="busuanzi_container_site_uv"><strong>: <a href=""><span id="busuanzi_value_site_uv"></span></strong></a> </span> &nbsp;&nbsp;
		  !--><span class="glyphicon glyphicon-eye-open"></span>&nbsp;&nbsp;<span id="busuanzi_container_site_pv"><strong>: <a href=""><span id="busuanzi_value_site_pv"></span></strong></a> </span>

		</div>
		<div class="col-sm-4">
		  Contact:<br />
		  Building B2, Guangzhou Laboratory<br />
		  96 Xingdao South Road, Guangzhou International Bio Island, <br />
		  Haizhu District, Guangzhou, Guangdong,<br />
		  China. <br />
          (<a href="https://goo.gl/maps/hDLr6siEAS2Eaypt8">Maps</a>)
		</div>
	  </div>
	</div>
  </div>
</div>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script src="https://www.ribocentre.org/js/bootstrap.min.js"></script>


  </body>

</html>
