---
layout: post
title:  "riboswitch_name"
data: 2023-03-27
author: Wenjian Liao
permalink: /docs/guanineold
---
**Rfam ID: [RF00167](https://rfam.org/family/RF00167)** <br>
&emsp;&emsp;&emsp;&emsp;**[]()** <br>


#### click into different sections:

- [Timeline](#timeline)
- [Description](#description)
- [Structure and Ligand recognition](#structure-and-ligand-recognition)
- [References](#references)

## Timeline
***
<html lang="zh-cn">
<div class="timeline">
  <div class="year">
    <div class="inner">
      <span>start</span>
    </div>
  </div>
  <ul class="days">
    <li class="day">
      <div class="events">
        <p style="width: 380px;">Discovery of xpt-pbuX operon</p>
        <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/9098051/" target="_blank">1997</a><sup>[1]</sup></div>
      </div>
    </li>
    <li class="day">
      <div class="events">
        <p style="padding-left: 30px;">Identification of guanine-I riboswitch in the xpt-pbuX 5'-UTR</p>
        <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/12787499/" target="_blank">2003</a><sup>[2]</sup></div>
      </div>
    </li>
    <li class="day">
      <div class="events">
        <p style="width: 380px;">Crystal structure of guanine-I riboswitch with hypoxanthine</p>
        <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/15549109/" target="_blank">2004</a><sup>[3]</sup></div>
      </div>
    </li>
    <li class="day">
      <div class="events">
        <p style="padding-left: 30px;">Crystal structure of guanine-I riboswitch with guanine</p>
        <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/15610857/" target="_blank">2004</a><sup>[4]</sup></div>
      </div>
    </li>
    <li class="day">
      <div class="events">
        <p style="width: 380px;">C74 is the sole determinant of guanine-I riboswitch specificity</p>
        <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/16650860/" target="_blank">2006</a><sup>[5]</sup></div>
      </div>
    </li>
    <li class="day">
      <div class="events">
        <p style="padding-left: 30px;">Ligand-binding capability of guanine-I riboswitch depends on Mg2+</p>
        <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/21890900/" target="_blank">2011</a><sup>[6]</sup></div>
      </div>
    </li>
    <li class="day">
      <div class="events">
        <p style="width: 380px;">Crystal structure of guanine-I riboswitch with an artificial nucleobase pair (Z:P pair)</p>
        <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/26223188/" target="_blank">2015</a><sup>[7]</sup></div>
      </div>
    </li>
    <li class="day">
      <div class="events">
        <p style="padding-left: 30px;">Discovery of guanine-II riboswitch</p>
        <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/35622895/" target="_blank">2015</a><sup>[8]</sup></div>
      </div>
    </li>
  </ul>
  <div class="year year--end">
    <div class="inner">
      <span>2023...</span>
    </div>
  </div>
</div>
</html>

## Description
***

<font size="4">The guanine riboswitch selectively recognizes guanine, and contains a cytosine ribonucleotide in a specific position of the guanine-binding aptamer domain. Now there are two classes of guanine riboswitches have been discovered, most commonly associated with genes encoding phosphoribosyltransferase (PRT) enzymes. Although these two classes of guanine riboswitches have the similar secondary structure, the directionalities of gene control upon ligand binding are predicted to be opposite for the guanine-I (OFF) and guanine-II (ON) riboswitches.</font>
<p><br /></p>

> Gene association
<font size="4">The guanine-I riboswitch, once thought to be an operon, was originally found in the 5’-UTR of xpt-pbuX gene. The xpt gene and pbuX gene encode a specific xanthine phosphoribosyl-transferase and a xanthine-specific purine permease respectively, and the genes have overlapping coding regions. The guanine-I riboswitch was also identified in other B. subtilis genes (purE, yxjA et.al.) and also in other bacterial species.[1][2]However, the nucleobase specificities for many annotated PRT enzymes downstream of guanine-II riboswitch have not been established<sup>[9]</sup>.</font>
<p><br /></p>
<table class="table table-bordered" style="table-layout:fixed;width:400px;margin-left:auto;margin-right:auto;">
<tr >
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/gene_association/riboswitch_guanine_gene_association1.svg" alt="drawing" style="width:400px;margin-top: 0px;margin-bottom: 0px;" >
</td>
</tr>
</table>
<p><br /></p>

> Gene regulation
<font size="4"><p>The guanine-I riboswitch has been shown to control gene expression through transcriptional termination. It forms an antiterminator in the absence of guanine, thereby allowing RNA transcription elongation to proceed to completion. The presence of guanine results in stabilization of the aptamer domain, thereby facilitating terminator formation and shutting down transcription<sup>[2][4]</sup>.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:800px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/gene_regulation/riboswitch_guanine_gene_regulation1.svg" alt="drawing" style="width:800px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
</tr>
</table>
<p><br /></p>
<font size="4"><p>Sequence and secondary structure model of a guanine-II riboswitch is consistent with a genetic “ON” switch wherein ligand binding would preclude formation of the terminator stem. Specifically, the ligand-bound aptamer structure requires that the orange-shaded nucleotides form a complete P1 structure, and the blue-shaded nucleotides form J3-1.<sup>[9]</sup>.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:400px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/gene_regulation/riboswitch_guanine_gene_regulation2.svg" alt="drawing" style="width:400px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
</tr>
</table>
<p><br /></p>
## Structure and Ligand recognition 
***


> 2D representation
<font size=4>The content for THFI </font><br>

<table class="table table-bordered" style="table-layout:fixed;width:700px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="style.css">
    <title>Document</title>
</head>
<style>
   body {
    width: 100%;
    height: 100vh;
}
   button {
   margin-right: 0px;
}
   .main-container {
    display: flex;
    align-items: left;
    justify-content: center;
    height: 100%;
}
   .zoom-wrapper1 {
    width: 350px;
    height: 350px;
    border: 1px solid #fff;
    display: flex;
    align-items: center;
    justify-content: center;
}
</style>
    <div class="main-container">
        <div class="zoom-wrapper1">
            <div class="zoom-area1">
                <img src="/images/2D/riboswitch_guanine_2D1.svg" alt="drawing" style="height:400px" />
            </div>
        </div>
    </div>
    <script src="https://www.ribocentre.org/js/panzoom.js"></script>
    <script type='text/javascript'>
      var zoomWraper1 = document.querySelector(".zoom-wrapper1");
      var panzoom1 = Panzoom(document.querySelector(".zoom-area1"), {
      maxScale: 6
      });
      zoomWraper1.addEventListener("wheel", panzoom1.zoomWithWheel);
      panzoom1.zoom(300 / document.querySelector(".zoom-area1 img").height);
      panzoom2.pan(0, 0);
      </script>
</td>
<!--2D模2--------------------------->
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
<style>
.zoom-wrapper2 {
    width: 350px;
    height: 350px;
    border: 1px solid #fff;
    display: flex;
    align-items: center;
    justify-content: center;
}
</style>
    <div class="main-container">
        <div class="zoom-wrapper2">
            <div class="zoom-area2">
                <img src="/images/2D/" alt="drawing" style="width:350px" />
            </div>
        </div>
    </div>
    <script src="https://www.ribocentre.org/js/panzoom.js"></script>
    <script type='text/javascript'>
      var zoomWraper2 = document.querySelector(".zoom-wrapper2");
      var panzoom2 = Panzoom(document.querySelector(".zoom-area2"), {
      maxScale: 6
      });
      zoomWraper2.addEventListener("wheel", panzoom2.zoomWithWheel);
      panzoom2.zoom(300 / document.querySelector(".zoom-area2 img").height);
    </script>
</td>
</tr></table><br>

<!--2D模-3-->
<font size=4>The content for THFII </font><br>
<table class="table table-bordered" style="table-layout:fixed;width:700px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
    <style>
        .zoom-wrapper3 {
            width: 350px;
            height: 350px;
            border: 1px solid #fff;
            display: flex;
            align-items: center;
            justify-content: center;
        }
    </style>
    <div class="main-container">
        <div class="zoom-wrapper3">
            <div class="zoom-area3">
                <img src="/images/2D/" alt="drawing" style="width:350px" />
            </div>
        </div>
    </div>
    <script src="https://www.ribocentre.org/js/panzoom.js"></script>
    <script type='text/javascript'>
      var zoomWraper3 = document.querySelector(".zoom-wrapper3");
      var panzoom3 = Panzoom(document.querySelector(".zoom-area3"), {
      maxScale: 6
      });
      zoomWraper3.addEventListener("wheel", panzoom3.zoomWithWheel);
      panzoom3.zoom(300 / document.querySelector(".zoom-area3 img").height);
    </script>
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <style>
  .zoom-wrapper4 {
      width: 450px;
      height: 400px;
      border: 1px solid #fff;
      display: flex;
      align-items: center;
      justify-content: center;
  }
  </style>
    <div class="main-container">
        <div class="zoom-wrapper4">
            <div class="zoom-area4">
                <img src="/images/2D/riboswitch_name_2D2.svg" alt="drawing" style="width:400px" />
            </div>
        </div>
    </div>
    <script src="https://www.ribocentre.org/js/panzoom.js"></script>
    <script type='text/javascript'>
      var zoomWraper4 = document.querySelector(".zoom-wrapper4");
      var panzoom4 = Panzoom(document.querySelector(".zoom-area4"), {
      maxScale: 6
      });
      zoomWraper4.addEventListener("wheel", panzoom4.zoomWithWheel);
      panzoom4.zoom(300 / document.querySelector(".zoom-area4 img").height);
    </script>
</td>
</tr></table><br>

<!--3D模块1-->
> 3D visualisation
<font size=4>The overall structure of the xpt-pbuX guanine riboswitch was generated from PDB ID:1Y27 at 2.40Å resolution bound with guanine. Guanine (shown in sticks) is labeled in red<sup>[4]</sup>. </font>
  <table class="table table-bordered" style="table-layout:fixed;width:700px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/riboswitch_guanine_3D1.svg" alt="drawing" style="width:350px;height:350px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="https://www.ebi.ac.uk/pdbe/pdb-component-library/js/pdbe-molstar-plugin-1.2.1.js"></script>
      <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .viewerSection {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:350px;
            height: 350px;
            position:relative;  
          }    
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
                  struct_asym_id:'A', 
                  start_residue_number:14, 
                  end_residue_number:21, 
                  color:{r:194,g:194,b:255}
                }
                ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 150px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 150px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        moleculeId: '1y27',
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }  
        

      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
    </script>
    </body>
    </html></td>
  </tr></table><br>
<p><br /></p>
> Binding pocket
<font size=4><p>Hydrogen-bonding alignments to bound guanine in the xpt-pbuX guanine riboswitch, generated from PDB ID:1Y27. The bound guanine forms a Watson-Crick pair with C74. Hydrogen bonds involving 2’-OH of U22 and base edges of U47 and U51 are common to both riboswitches<sup>[4]</sup>. <br /></p>
</font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/binding_pockets/riboswitch_guanine_binding_pocket1.svg" alt="drawing" style="width:400px"  px="" /></td>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/binding_pockets/riboswitch_guanine_binding_pocket2.svg" alt="drawing" style="width:400px"  px="" /></td>
</tr></table>
<p><br /></p>


> Ligand recognition
<font size=4>
<p>Chemical structures of guanine and its various analogs. The apparent KD of each compound of guanine-I riboswitch and partial compounds of guanine-II riboswitch are shown on bottom. G-I, guanine-I riboswitch. G-II, guanine-II riboswitch<sup>[2,9]</sup>.<br /><br /></p>
</font>
<table class="table table-bordered" style="table-layout:fixed;width:400px;margin-left:auto;margin-right:auto;">
<tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/ligand_recognition/riboswitch_guanine_ligand_recognition1.svg" alt="drawing" style="width:400px;margin-top: 0px;margin-bottom: 0px;" >
</td>
</tr>
</table>
<p><br /></p>




## References
***

**[1] Xanthine metabolism in Bacillus subtilis: characterization of the xpt-pbuX operon and evidence for purine- and nitrogen-controlled expression of genes involved in xanthine salvage and catabolism.**<br>
L. C. Christiansen, S. Schou, P. Nygaard and H. H. Saxild <br>
[Journal of Bacteriology 179 (8): 2540-2550.(1997)](https://pubmed.ncbi.nlm.nih.gov/20659680/)<br><br>

**[2] Riboswitches Control Fundamental Biochemical Pathways in Bacillus subtilis and Other Bacteria.**<br>
M. Mandal, B. Boese, J. E. Barrick, W. C. Winkler and R. R. Breaker <br>
[ Cell 113 (5): 577-586.(2003)](https://pubmed.ncbi.nlm.nih.gov/12787499/)<br><br>

**[3] Structure of a natural guanine-responsive riboswitch complexed with the  metabolite hypoxanthine.**<br>
R. T. Batey, S. D. Gilbert and R. K. Montange  <br>
[Nature 432 (7015): 411-5.(2004)](https://pubmed.ncbi.nlm.nih.gov/15549109/)<br><br>

**[4] Structural Basis for Discriminative Regulation of Gene Expression by Adenine- and Guanine-Sensing mRNAs.**<br>
A. Serganov, Y. Yuan, O. Pikovskaya, A. Polonskaia, L. Malinina, A. T. Phan, C. Hobartner, R. Micura, R. R. Breaker and D. J. Patel  <br>
[Chemistry & Biology 11 (12): 1729-1741.(2004)](https://pubmed.ncbi.nlm.nih.gov/15610857/)<br><br>

**[5] Thermodynamic and Kinetic Characterization of Ligand Binding to the Purine Riboswitch Aptamer Domain.**<br>
S. D. Gilbert, C. D. Stoddard, S. J. Wise and R. T. Batey<br>
[Journal of Molecular Biology 359 (3): 754-768.(2006)](https://pubmed.ncbi.nlm.nih.gov/16650860/)<br><br>

**[6] Adaptive Ligand Binding by the Purine Riboswitch in the Recognition of Guanine and Adenine Analogs.**<br>
S. D. Gilbert, F. E. Reyes, A. L. Edwards and R. T. Batey <br>
[Structure 17 (6): 857-868.(2009)](https://pubmed.ncbi.nlm.nih.gov/19523903/)<br><br>

**[7] Influence of ground-state structure and Mg 2+ binding on folding kinetics of the guanine-sensing riboswitch aptamer domain.**<br>
J. Buck, A. Wacker, E. Warkentin, J. Wöhnert, J. Wirmer-Bartoschek and H. Schwalbe <br>
[Nucleic Acids Research 39 (22): 9768-9778.(2011)](https://pubmed.ncbi.nlm.nih.gov/21890900/)<br><br>

**[8] A Crystal Structure of a Functional RNA Molecule Containing an Artificial Nucleobase Pair.**<br>
A. R. Hernandez, Y. Shao, S. Hoshika, Z. Yang, S. A. Shelke, J. Herrou, H. J. Kim, M. J. Kim, J. A. Piccirilli and S. A. Benner <br>
[Angewandte Chemie 127 (34): 9991-9994.(2015)](https://pubmed.ncbi.nlm.nih.gov/26223188/)<br><br>

**[9] Variants of the guanine riboswitch class exhibit altered ligand specificities for xanthine, guanine, or 2′-deoxyguanosine.**<br>
S. Hamal Dhakal, S. S. S. Panchapakesan, P. Slattery, A. Roth and R. R. Breaker <br>
[Proceedings of the National Academy of Sciences ,119 (22).(2022)](https://pubmed.ncbi.nlm.nih.gov/35622895/)<br><br>

