---
layout: post
title:  "Lysine"
date:   2023-04-23 00:00:00
author: Yuanyin Huang, Xiaowei Lin
permalink: /docs/Lysine/
---
<p style="font-size: 24px"><strong>Rfam ID: [RF00168](https://rfam.xfam.org/family/RF00168,target="_blank")(Lysine riboswitch)</strong> <br /></p>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
  .header_box {
    border: none;
    border-bottom: 4px solid #005826;
    box-shadow: 5px 5px 5px 5px rgba(0, 0, 0, 0.25);
    font-size:32px
  }
</style>
</head>
<p><br /></p>
<p class="header_box">Timeline</p>
<p><br /></p>
<div class="timeline">
  <div class="year">
    <div class="inner">
      <span>start</span>
    </div>
  </div>
  <ul class="days">
          
    <li class="day">
     <div class="events">
       <p >Discovery of lysine riboswitch</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/14597663/" target="_blank">2003</a><sup>[1]</sup></div>
     </div>
    </li>
        

    <li class="day">
     <div class="events">
       <p >Discovery of lysine riboswitch</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/14523230/" target="_blank">2003</a><sup>[2]</sup></div>
     </div>
    </li>
        

    <li class="day">
     <div class="events">
       <p >A loop–loop interaction and a K-turn motif are important</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/17585050/" target="_blank">2007</a><sup>[3]</sup></div>
     </div>
    </li>
        

    <li class="day">
     <div class="events">
       <p >Several lysine analogs binding riboswitches in vitro inhibit Bacillus subtilis growth</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/17143270/" target="_blank">2007</a><sup>[4]</sup></div>
     </div>
    </li>
        

    <li class="day">
     <div class="events">
       <p >Structure of lysine riboswitch</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/18593706/" target="_blank">2008</a><sup>[5]</sup></div>
     </div>
    </li>
        

    <li class="day">
     <div class="events">
       <p >Structure of lysine riboswitch</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/18784651/" target="_blank">2008</a><sup>[6]</sup></div>
     </div>
    </li>
        

    <li class="day">
     <div class="events">
       <p >The interaction of Helix P2–loop L4 is particularly important</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/21169337/" target="_blank">2011</a><sup>[7]</sup></div>
     </div>
    </li>
        

    <li class="day">
     <div class="events">
       <p >The lysine riboswitch is highly sensitive to the precise placement of the ε-amino group</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/22771573/" target="_blank">2012</a><sup>[8]</sup></div>
     </div>
    </li>
        

    <li class="day">
     <div class="events">
       <p >Rationally-designed fluorescent lysine riboswitch probes</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/22961337/" target="_blank">2012</a><sup>[9]</sup></div>
     </div>
    </li>
        

    <li class="day">
     <div class="events">
       <p >The structural elements of the lysine molecule required for recognition</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/22416067/" target="_blank">2012</a><sup>[10]</sup></div>
     </div>
    </li>
        

    <li class="day">
     <div class="events">
       <p >The engineering of a lysine-ON riboswitch for metabolic control in Corynebacterium glutamicum</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/26300047/" target="_blank">2015</a><sup>[11]</sup></div>
     </div>
    </li>
        

    <li class="day">
     <div class="events">
       <p >The use of lysine riboswitch binding L-lysine to control the competing metabolic bypathways of lysine biosynthesis</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/25575181/" target="_blank">2015</a><sup>[12]</sup></div>
     </div>
    </li>
        

    <li class="day">
     <div class="events">
       <p >Lysine binding residues in the global folding of the lysC riboswitch is important</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/26403229/" target="_blank">2015</a><sup>[13]</sup></div>
     </div>
    </li>
        

    <li class="day">
     <div class="events">
       <p >Trimethylamine N-oxide (TMAO) can stabilize the lysine riboswitch</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/32706360/" target="_blank">2020</a><sup>[15]</sup></div>
     </div>
    </li>
        

    <li class="day">
     <div class="events">
       <p >K<sup>+</sup> is optimally suited for bridging interactions between lysine and the riboswitch</p>
       <div class="date"><a href="https://pubmed.ncbi.nlm.nih.gov/36916791/" target="_blank">2023</a><sup>[16]</sup></div>
     </div>
    </li>
        

  </ul>
  <div class="year year--end">
    <div class="inner">
      <span>2023...</span>
    </div>
  </div>
</div>
</html>
        
<p class="header_box">Description</p>
***
<font size="4">The Lysine riboswitch is a metabolite binding RNA element found within certain messenger RNAs that serve as a precision sensor for the amino acid lysine. Allosteric rearrangement of mRNA structure is mediated by ligand binding, and this results in modulation of gene expression. Lysine riboswitch are most abundant in Bacillota and Gammaproteobacteria where they are found upstream of a number of genes involved in lysine biosynthesis, transport and catabolism. The lysine riboswitch has also been identified independently and called the L box<sup></sup>.</font>
<p><br /></p>
            
> Gene association
        
<font size="4">The pathway of lysine riboswitches regulating lysine synthesis<sup>[2]</sup>.</font>
<p><br /></p>
<table class="table table-bordered" style="table-layout:fixed;width:800px;margin-left:auto;margin-right:auto;">
<tr >
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/gene_association/Lysine_riboswitch_gene_association.svg" alt="drawing" style="width:800px;margin-top: 0px;margin-bottom: 0px;" >
</td>
</tr>
</table>
<p><br /></p>
                
> Gene regulation
            
<font size="4"><p>Potential mechanism of translation regulation by lysine riboswitch<sup>[2]</sup>.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:800px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/gene_regulation/Lysine_riboswitch_gene_regulation.svg" alt="drawing" style="width:800px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
</tr>
</table>
<p><br /></p>
                    
<p class="header_box">Structure and Ligand recognition</p>
> 2D representation
        
<font size=4>(Left) Consensus sequence and secondary structure model for lysine riboswitch. (Right) Secondary structure depictions of  lysine riboswitch according to PDB ID:3DIL<sup>[1,6]</sup>.</font><br>
<table class="table table-bordered" style="table-layout:fixed;width:800px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="style.css">
    <title>Document</title>
</head>
<style>
   body {
    width: 100%;
    height: 100vh;
}
   button {
   margin-right: 0px;
}
   .main-container {
    display: flex;
    align-items: left;
    justify-content: center;
    height: 100%;
}
   .zoom-wrapper1 {
    width: 400px;
    height: 400px;
    border: 1px solid #fff;
    display: flex;
    align-items: center;
    justify-content: center;
}
</style>
    <div class="main-container">
        <div class="zoom-wrapper1">
            <div class="zoom-area1">
                <img src="/images/2D/Lysine_riboswitch_2D1.svg" alt="drawing" style="width:400px" />
            </div>
        </div>
    </div>
    <script src="https://www.ribocentre.org/js/panzoom.js"></script>
    <script type='text/javascript'>
      var zoomWraper1 = document.querySelector(".zoom-wrapper1");
      var panzoom1 = Panzoom(document.querySelector(".zoom-area1"), {
      maxScale: 6
      });
      zoomWraper1.addEventListener("wheel", panzoom1.zoomWithWheel);
      panzoom1.zoom(300 / document.querySelector(".zoom-area1 img").height);
      </script>
</td>
<!--2D模2--------------------------->
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
<style>
.zoom-wrapper2 {
    width: 400px;
    height: 400px;
    border: 1px solid #fff;
    display: flex;
    align-items: center;
    justify-content: center;
}
</style>
    <div class="main-container">
        <div class="zoom-wrapper2">
            <div class="zoom-area2">
                <img src="/images/2D/Lysine_riboswitch_2D2.svg" alt="drawing" style="width:400px" />
            </div>
        </div>
    </div>
    <script src="https://www.ribocentre.org/js/panzoom.js"></script>
    <script type='text/javascript'>
      var zoomWraper2 = document.querySelector(".zoom-wrapper2");
      var panzoom2 = Panzoom(document.querySelector(".zoom-area2"), {
      maxScale: 6
      });
      zoomWraper2.addEventListener("wheel", panzoom2.zoomWithWheel);
      panzoom2.zoom(300 / document.querySelector(".zoom-area2 img").height);
    </script>
</td>
</tr></table><br>
                
> 3D visualisation
            
<font size=4>The overall structure of lysine riboswitch was generated from PDB ID:3DIL at 1.90 Å resolution bound with lysine.  Lysine (shown in sticks) is labeled in red <sup>[6]</sup>.</font>
  <table class="table table-bordered" style="table-layout:fixed;width:800px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/Lysine_riboswitch_3D1.svg" alt="drawing" style="width:400px;height:400px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="https://www.ebi.ac.uk/pdbe/pdb-component-library/js/pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:400px;
            height: 400px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A', 
  start_residue_number:1, 
  end_residue_number:11, 
  color:{r:128,g:219,b:158}
},
{
  struct_asym_id: 'A', 
  start_residue_number:12, 
  end_residue_number:79, 
  color:{r:10,g:11,b:255}
},
{
  struct_asym_id: 'A', 
  start_residue_number:80, 
  end_residue_number:113, 
  color:{r:255,g:223,b:92}
},
{
  struct_asym_id: 'A', 
  start_residue_number:114, 
  end_residue_number:140, 
  color:{r:255,g:170,b:220}
},
{
  struct_asym_id: 'A', 
  start_residue_number:141, 
  end_residue_number:162, 
  color:{r:9,g:254,b:254}
},
{
  struct_asym_id: 'A', 
  start_residue_number:163, 
  end_residue_number:174, 
  color:{r:128,g:219,b:158}
},
{
  struct_asym_id: 'A', 
  start_residue_number:175, 
  end_residue_number:175, 
  color:{r:255,g:11,b:12}
},
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/docs/pdbfiles/3DIL.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
    </script>
    </body>
    </html></td>
  </tr></table><br>
                
> Binding pocket
                
<font size=4><p>(Left) Surface representation of the binding pocket of lysine riboswitch generated from PDB ID:3DIL. Lysine (shown in sticks) is labeled in red. (Right) The hydrogen bonds of the binding site of lysine  riboswitch bound with lysine<sup>[6]</sup>.<br /></p></font>
<table class="table table-bordered" style="table-layout:fixed;width:800px;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/binding_pockets/Lysine_riboswitch_binding_pockets1.svg" alt="drawing" style="width:400px"  px="" /></td>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/binding_pockets/Lysine_riboswitch_binding_pockets2.svg" alt="drawing" style="width:400px"  px="" /></td>
</tr>
</table>
                    
> Ligand recognition
            
<font size=4>
<p>Chemical structures of lysine and its analogs. The apparent KD of each compound of lysine riboswitch is shown on bottom<sup>[1,4]</sup>.<br /><br /></p>
</font>
<table class="table table-bordered" style="table-layout:fixed;width:800px;margin-left:auto;margin-right:auto;">
<tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/ligand_recognition/Lysine_riboswitch_ligand_recognition.svg" alt="drawing" style="width:800px;margin-top: 0px;margin-bottom: 0px;" >
</td>
</tr>
</table>
<p><br /></p>
                
<p class="header_box">References</p>
<p><br /></p>
            
<p><strong>[1]An mRNA structure in bacteria that controls gene expression by binding lysine</strong><br />
Sudarsan, N., Wickiser, J. K., Nakamura, S., Ebert, M. S. & Breaker, R. R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/14597663/" target="_blank">Genes Dev. 17, 2688–2697 (2003).</a><br /><br /></p>
                
<p><strong>[2]The L box regulon: lysine sensing by leader RNAs of bacterial lysine biosynthesis genes</strong><br />
Grundy, F. J., Lehman, S. C. & Henkin, T. M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/14523230/" target="_blank">Proc. Natl. Acad. Sci. U. S. A. 100, 12057–12062 (2003).</a><br /><br /></p>
                
<p><strong>[3]A loop loop interaction and a K-turn motif located in the lysine aptamer domain are important for the riboswitch gene regulation control</strong><br />
Blouin, S. & Lafontaine, D. A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/17585050/" target="_blank">RNA 13, 1256–1267 (2007).</a><br /><br /></p>
                
<p><strong>[4]Antibacterial lysine analogs that target lysine riboswitches</strong><br />
Blount, K. F., Wang, J. X., Lim, J., Sudarsan, N. & Breaker, R. R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/17143270/" target="_blank">Nat. Chem. Biol. 3, 44–49 (2007).</a><br /><br /></p>
                
<p><strong>[5]Crystal structure of the lysine riboswitch regulatory mRNA element</strong><br />
Garst, A. D., Héroux, A., Rambo, R. P. & Batey, R. T.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18593706/" target="_blank">J. Biol. Chem. 283, 22347–22351 (2008).</a><br /><br /></p>
                
<p><strong>[6]Structural insights into amino acid binding and gene control by a lysine riboswitch</strong><br />
Serganov, A., Huang, L. & Patel, D. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18784651/" target="_blank">Nature 455, 1263–1267 (2008).</a><br /><br /></p>
                
<p><strong>[7]Folding of the lysine riboswitch: importance of peripheral elements for transcriptional regulation</strong><br />
Blouin, S., Chinnappan, R. & Lafontaine, D. A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/21169337/" target="_blank">Nucleic Acids Res. 39, 3373–3387 (2011).</a><br /><br /></p>
                
<p><strong>[8]Insights into the regulatory landscape of the lysine riboswitch</strong><br />
Garst, A. D., Porter, E. B. & Batey, R. T.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/22771573/" target="_blank">J. Mol. Biol. 423, 17–33 (2012).</a><br /><br /></p>
                
<p><strong>[9]Rationally-designed fluorescent lysine riboswitch probes</strong><br />
Budhathoki, P., Bernal-Perez, L. F., Annunziata, O. & Ryu, Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/22961337/" target="_blank">Org. Biomol. Chem. 10, 7872–7874 (2012).</a><br /><br /></p>
                
<p><strong>[10]Analysis of lysine recognition and specificity of the Bacillus subtilis L box riboswitch</strong><br />
Wilson-Mitchell, S. N., Grundy, F. J. & Henkin, T. M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/22416067/" target="_blank">Nucleic Acids Res. 40, 5706–5717 (2012).</a><br /><br /></p>
                
<p><strong>[11]Engineering a Lysine-ON Riboswitch for Metabolic Control of Lysine Production in Corynebacterium glutamicum</strong><br />
Zhou, L.-B. & Zeng, A.-P.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/26300047/" target="_blank">ACS Synth. Biol. 4, 1335–1340 (2015).</a><br /><br /></p>
                
<p><strong>[12]Exploring lysine riboswitch for metabolic flux control and improvement of L-lysine synthesis in Corynebacterium glutamicum</strong><br />
Zhou, L.-B. & Zeng, A.-P.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25575181/" target="_blank">ACS Synth. Biol. 4, 729–734 (2015).</a><br /><br /></p>
                
<p><strong>[13]Role of lysine binding residues in the global folding of the lysC riboswitch</strong><br />
Smith-Peter, E., Lamontagne, A.-M. & Lafontaine, D. A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/26403229/" target="_blank">RNA Biol. 12, 1372–1382 (2015).</a><br /><br /></p>
                
<p><strong>[14]Comparative genomics and phylogenomic analyses of lysine riboswitch distributions in bacteria</strong><br />
Mukherjee, S., Barash, D. & Sengupta, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/28873470/" target="_blank">PLoS One 12, e0184314 (2017).</a><br /><br /></p>
                
<p><strong>[15]High pressure single-molecule FRET studies of the lysine riboswitch: cationic and osmolytic effects on pressure induced denaturation</strong><br />
Sung, H.-L. & Nesbitt, D. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/32706360/" target="_blank">Phys. Chem. Chem. Phys. 22, 15853–15866 (2020).</a><br /><br /></p>
                
<p><strong>[16]Ionic Cooperativity between Lysine and Potassium in the Lysine Riboswitch: Single-Molecule Kinetic and Thermodynamic Studies</strong><br />
Marton Menendez, A. & Nesbitt, D. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/36916791/" target="_blank">J. Phys. Chem. B 127, 2430–2440 (2023).</a><br /><br /></p>
                